diff -urN gtoaster.orig/po/ca.po gtoaster/po/ca.po
--- gtoaster.orig/po/ca.po	Sun Jun  8 20:13:20 2003
+++ gtoaster/po/ca.po	Sun Jun  8 21:35:15 2003
@@ -6,13 +6,13 @@
 #, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
+"Project-Id-Version: 1.0Beta6\n"
 "POT-Creation-Date: 2002-08-28 08:50+0200\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
+"PO-Revision-Date: 2002-08-28 08:50+0200\n"
+"Last-Translator: Unknown <unknown@not-assigned.org>\n"
+"Language-Team: Catalan\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=CHARSET\n"
+"Content-Type: text/plain; charset=iso-8859-1\n"
 "Content-Transfer-Encoding: 8bit\n"
 
 #: audio.c:46
diff -urN gtoaster.orig/po/ca.po~ gtoaster/po/ca.po~
--- gtoaster.orig/po/ca.po~	Thu Jan  1 01:00:00 1970
+++ gtoaster/po/ca.po~	Wed Aug 28 08:55:19 2002
@@ -0,0 +1,1707 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"POT-Creation-Date: 2002-08-28 08:50+0200\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=CHARSET\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: audio.c:46
+msgid "Audio System"
+msgstr "Sistema d'udio"
+
+#: audio.c:50
+msgid "Audio driver"
+msgstr "Controlador d'udio"
+
+#: audiotrack.c:247
+#, c-format
+msgid "CD %#x Audiotrack %02d"
+msgstr "CD %#x pista %02d"
+
+#: audiotrack.c:251
+#, c-format
+msgid "CD in %s Audiotrack %02d"
+msgstr "CD en %s pista %02d"
+
+#: audiotrack.c:302
+msgid "CD Digital Audio"
+msgstr "CD Digital d'udio"
+
+#: audiotrack.c:306
+msgid "Audio track Client:"
+msgstr "Programa per extraure audio:"
+
+#: audiotrack.c:311
+msgid "Audio track reading:"
+msgstr "Lectura de pistes d'udio:"
+
+#: audiotrack.c:316
+msgid "CD Text retrieval:"
+msgstr "extracci CD Text:"
+
+#: audiotrack.c:320
+msgid "CD Text parse Title:"
+msgstr "analizador de ttol CD Text:"
+
+#: audiotrack.c:324
+msgid "CD Text parse Artist:"
+msgstr "CD Text, analitzar Artista:"
+
+#: audiotrack.c:328
+msgid "CD Text parse Disc Title:"
+msgstr "CD Text,analitzar el Ttol del Disc:"
+
+#: audiotrack.c:332
+msgid "CD Text parse Disc Artist:"
+msgstr "CD Text, analitzar Autor del disc:"
+
+#: audiotrack.c:337
+msgid "Precache Audio Tracks:"
+msgstr ""
+
+#: cddb.c:892
+msgid "CDDB server"
+msgstr "servidor CDDB"
+
+#: cddrives.c:160
+msgid "Drive Type"
+msgstr "Tipus de dispositiu"
+
+#: cddrives.c:161
+msgid "Manufacturer Name"
+msgstr "Nom del fabricant"
+
+#: cddrives.c:162
+msgid "Device Name"
+msgstr "Nom del dispositiu"
+
+#: cddrives.c:163 cddrives.c:436 cddrives.c:491
+msgid "Scsi ID"
+msgstr "Scsi ID"
+
+#: cddrives.c:164 cddrives.c:440 cddrives.c:495
+msgid "Mountpoint"
+msgstr "Punt de montatge"
+
+#: cddrives.c:422 cddrives.c:477
+msgid "Please enter drive data"
+msgstr "Introdur informaci del Dispossitiu"
+
+#: cddrives.c:426 cddrives.c:481 cddrives.c:1074
+msgid "Model"
+msgstr "Model"
+
+#: cddrives.c:429 cddrives.c:484 cddrives.c:1075
+msgid "Manufacturer"
+msgstr "Fabricant"
+
+#: cddrives.c:433 cddrives.c:488
+msgid "Device File"
+msgstr "Fitxer de dispossitiu"
+
+#: cddrives.c:444 cddrives.c:499
+msgid "This Drive is a CD writer"
+msgstr "Aquesta unitat es un gravador de CD"
+
+#: cddrives.c:448 cddrives.c:503
+msgid "Use SCSI Interface for DAE"
+msgstr "Emprar interfcie SCSI per a DAE"
+
+#: cddrives.c:458 cdsource.c:119 cddrives.c:513
+msgid "(Model)"
+msgstr "(Model)"
+
+#: cddrives.c:459 cddrives.c:514
+msgid "(Manufacturer)"
+msgstr "(Fabricant)"
+
+#: cddrives.c:460 cddrives.c:515
+msgid "/dev/scd0"
+msgstr "/dev/scd0"
+
+#: cddrives.c:461 cddrives.c:516
+msgid "0,0"
+msgstr "0,0"
+
+#: cddrives.c:462 cddrives.c:517
+msgid "/mnt/cdrom"
+msgstr "/mnt/cdrom"
+
+#: cddrives.c:556 cddrives.c:611
+msgid "CDROM and Recorder Setup"
+msgstr "Configuraci del CDROM i Gravadora"
+
+#: cddrives.c:578 cddrives.c:633
+msgid "Add"
+msgstr "Afegir"
+
+#: cddrives.c:583 filetypes.c:412 cddrives.c:638
+msgid "Remove"
+msgstr "Eliminar"
+
+#: cddrives.c:588 cddrives.c:643 menusys.h:72
+msgid "Edit"
+msgstr "Editar"
+
+#: cddrives.c:593 cddrives.c:648
+msgid "Scan Bus"
+msgstr "Explorar Bus"
+
+#: cddrives.c:601 cddrives.c:656
+msgid "Perform bus scan at startup"
+msgstr "Explorar Bus en iniciar"
+
+#: configupdate.c:253 configupdate.c:266
+msgid ""
+"This Version of Gnometoaster is providing a new\n"
+"drive setup scheme. You have to reconfigure your\n"
+"CDROM and recorder settings. Usually, all it takes\n"
+"is pressing the \"Scan Bus\" button in Gnometoaster's\n"
+"new drive setup\n"
+msgstr ""
+"Aquesta versi de Gnometoaster est provista d'un\n"
+"nou esquema de configuraci de dispositius. Has de\n"
+"reconfigurar els parmetres del CDROM i la gravadora.\n"
+"Normalment, sols has de premer el bot \"Explorar Bus\"\n"
+"en la configuraci de Dispositiu Nou.\n"
+
+#: configupdate.c:443 configupdate.c:456
+msgid ""
+"A Configfile for a newer version\n"
+"of Gnometoaster was found.\n"
+"Continuing may be dangerous because\n"
+"of possible changes in the file\n"
+"structure which may destroy your\n"
+"entire configuration.\n"
+"Do you want to run Gnometoaster anyway ?\n"
+msgstr ""
+"S'ha trobat un arxiu de configuraci per\n"
+"a una nova versi de Gnometoaster.\n"
+"Continuar pot ser perills a causa dels\n"
+"possibles canvis en l'estructura que\n"
+"poden destruir per complet la seva\n"
+"configuraci.\n"
+"Vols executar de totes maneres Gnometoaster?\n"
+
+#: configupdate.c:451 configupdate.c:464
+msgid ""
+"Introducing multisession toc and dummy write\n"
+"functionality into gnometoaster requires a few\n"
+"modifications in gnometoaster's configuration setup\n"
+"I can do the necessary modifications\n"
+"automatically.You will lose\n"
+"your customized configuration\n"
+"of \"Preferences/Recorder/Trackwrite command\",\n"
+"\"Preferences/Recorder/Fixating Command\"\n"
+"and \"Preferences/Recorder/Clear Disc\"\n"
+"If you want to modify those entries yourself,\n"
+"just answer NO to this question.\n"
+"(New Variables rec_multisession,rec_dummymode)\n"
+msgstr ""
+"La introducci de TOC multissessi i escritura\n"
+"de prova en gnometoaster requireix unes poques\n"
+"modificacions en els parmetres de configuraci\n"
+"Puc fer les modificacions necessaries automticament.\n"
+"Perdrs la teva configuraci personalitzada de \n"
+"\"Preferncies/Gravadora/Parmetres de escritura de pistes,\n"
+"\"Preferncies/Gravadora/Parmetres per a fixar\"\n"
+"i \"Preferncies/Gravadora/Borrar disc\"\n"
+"Si vols modificar aquests registres per tu mateix,\n"
+"respon NO a auesta pregunta.\n"
+"(Variables noves rec_multisession,rec_dummymode)\n"
+
+#: configupdate.c:464 configupdate.c:477
+msgid ""
+"This version of Gnometoaster switches the underlying\n"
+"filesystem creator from mkhybrid to mkisofs.\n"
+"You can,of course,continue to use mkhybrid.\n"
+"Just answer no to this question in this case.\n"
+msgstr ""
+
+#: configupdate.c:469 configupdate.c:482
+msgid ""
+"This update fixes a bug in the mp3 client call\n"
+"which made gnometoaster record non-cd-quality\n"
+"mp3 files with 2x or 4x the speed of the original.\n"
+"(modifies filetype registry's mp3 entry).\n"
+"\n"
+"Also,El Torito bootable support requires your mkisofs\n"
+"calls to be modified.\n"
+msgstr ""
+
+#: configupdate.c:476 configupdate.c:489
+msgid ""
+"This version of Gnometoaster enables setting the\n"
+"volume id of an iso track from fsedit.\n"
+"To accomplish this,the default mkisofs call\n"
+"in both the standard and the multisession version.\n"
+"has to be modified.\n"
+msgstr ""
+
+#: configupdate.c:482 configupdate.c:495
+msgid ""
+"This version of Gnometoaster makes it possible to calculate\n"
+"the tracksize of an isotrack accurately by calling mkisofs\n"
+"for that purpose. Some further cleanups rid the configuration\n"
+"of a separate multisession isotrack call,incorporating multi-\n"
+"session support into the main iso image call as an option.\n"
+"One of the immediate results of those efforts is that now\n"
+"we can safely give the calculated tracksize to cdrecord\n"
+"to support recorders needing this value in advance.\n"
+"However,to accomplish that,the cdrecord write_track call\n"
+"has to be updated.\n"
+msgstr ""
+
+#: configupdate.c:493 configupdate.c:506
+msgid ""
+"This version of Gnometoaster is capable of calculating the\n"
+"Tracksize of an audio track created from a wav file accurately.\n"
+"To make use of this feature,the tracksize detection line\n"
+"of the wav file filetype registry entry needs to be changed.\n"
+msgstr ""
+
+#: configupdate.c:498 configupdate.c:511
+msgid ""
+"In order to fix a bug calculating the tracksize of an El Torito\n"
+"bootable filesystem,Gnometoaster needs to update the calculation\n"
+"rules stored in isotrack_tscalc.\n"
+msgstr ""
+
+#: configupdate.c:503 configupdate.c:516
+msgid ""
+"This fix removes the -a option from the default mkisofs call\n"
+"to meet the requirements of recent mkisofs versions which would\n"
+"otherwise reject the call and create wrong track sizes for\n"
+"iso tracks.\n"
+"Also, several calls must be\n"
+"updated to match the new drive setup scheme.\n"
+"Apart from that, a bug recording/playing mono mp3s has been\n"
+"fixed in the default mpg123 call.\n"
+"Finally, from this version on gnometoaster can calculate the\n"
+"tracksize of mp3 audio tracks internally.\n"
+"Your configfile will be changed to facilitate this new\n"
+"functionality.\n"
+msgstr ""
+
+#: configupdate.c:516 configupdate.c:529
+msgid ""
+"We have do makde a few adjustments to the standard DAE call\n"
+"in Gnometoaster's preferences in order to support an additional\n"
+"field of Gnometoaster's drive setup scheme.\n"
+msgstr ""
+
+#: configupdate.c:520 configupdate.c:533
+msgid ""
+"This version of Gnometoaster introduces the capability to store\n"
+"audio tracks to your harddrive as files. Gnometoaster will\n"
+"now adjust it's preferences to support storing audio tracks\n"
+"to your harddrive as either .wav or .mp3 files (where the latter\n"
+"is the default).\n"
+msgstr ""
+
+#: configupdate.c:526 configupdate.c:539
+msgid ""
+"This version fixes a wrong sox call for audiotrack->wav conversion.\n"
+"in your filetype registry.\n"
+msgstr ""
+
+#: configupdate.c:529 configupdate.c:542
+msgid ""
+"This version of Gnometoaster fixes a bug in the mountpoint autodetect\n"
+"expression of your recorder setup\n"
+"If you're running on a Big Endian system (e.g. PowerPC) I'll fix\n"
+"some parameters for the mp3 decoder to work properly\n"
+msgstr ""
+
+#: configupdate.c:534 configupdate.c:547
+msgid ""
+"This version will fix broken .wav functionality on Big Endian Systems (e.g. "
+"PowerPC).\n"
+"It also removes the option to configure the filesystem destination area from "
+"your preferences.\n"
+"The symbolic link tree will be built in your temp directory from now on.\n"
+"Also note that the default cdrdao call will be changed to support CD-Text "
+"writing which implies making \"generic-mmc\" it's default driver\n"
+msgstr ""
+
+#: configupdate.c:540 configupdate.c:553
+msgid ""
+"Due to changes in the commandline parameters of mkisofs in recent releases\n"
+"Gnometoaster's tracksize detection scheme for ISO tracks needs to be "
+"changed.\n"
+msgstr ""
+
+#: configupdate.c:543 configupdate.c:556
+msgid ""
+"This version of GnomeToaster introduces two new filetypes supported by "
+"default:\n"
+"Ogg-Vorbis and Shorten. I will now create the corresponding filetype "
+"registry entries\n"
+"for those filetypes. If you want to actually use them, also make sure to "
+"install the commandline tools\n"
+"needed to decode them. GnomeToaster will tell you about the tools you need "
+"when you start using\n"
+"the new filetypes. (Note: In case you already have entries for .ogg or .shn "
+"in your filetypes registry, those entries will be left untouched).\n"
+"This version of GnomeToaster also adds native support for buffer-underrun "
+"protection schemes of all kind (burn-proof, just-link, ...) as far as "
+"supported by cdrecord."
+msgstr ""
+
+#: configupdate.c:552 configupdate.c:565
+msgid ""
+"This version of GnomeToaster will automatically set the title and artist ID3 "
+"tags\n"
+"of any mp3 files encoded through it using the mp3info tool.\n"
+"This requires an additional commandline call to be inserted into the mp3 "
+"encoder profile\n"
+"of GnomeToaster's internal filetypes registry.\n"
+"Do you want me to perform all necessary changes for you ?\n"
+msgstr ""
+
+#: configupdate.c:557 configupdate.c:570
+msgid ""
+"This version of GnomeToaster adapts to a new commandline scheme to be "
+"applied\n"
+"for ogg123, the ogg-vorbis file decoder.\n"
+"Answering this question with yes will break compatibility with\n"
+"versions of ogg123 older than 1.0rc2\n"
+msgstr ""
+
+#: configupdate.c:564 configupdate.c:581
+msgid ""
+"You're running Gnometoaster for the first time.\n"
+"All Preferences have been set to default values.\n"
+"Make sure you change all settings to match your system!"
+msgstr ""
+"Es la primera vegara que executes GnomeToaster.\n"
+"Totes les Preferncies estan en el seu valor Predeterminat.\n"
+"Assegurat de canviar les opcions per a que et funcione correctamnet!"
+
+#: configupdate.c:641 configupdate.c:659
+msgid "Configfile maintenance informations"
+msgstr ""
+
+#: datacopydlg.c:123
+msgid "Throughput:"
+msgstr "Rendiment Total:"
+
+#: datacopydlg.c:127
+msgid " kb/sec.\n"
+msgstr " kb/sec.\n"
+
+#: datacopydlg.c:135
+msgid " elapsed\n"
+msgstr "Temps transcorregut\n"
+
+#: datacopydlg.c:147
+msgid " remaining\n"
+msgstr "Restant\n"
+
+#: datacopydlg.c:150
+msgid "please wait...\n"
+msgstr "Espera ...\n"
+
+#: datacopydlg.c:172
+#, c-format
+msgid "(%.0f%% done) %s"
+msgstr "(%.0f%% aconseguit) %s"
+
+#: datacopydlg.c:287
+msgid "please wait..."
+msgstr "Espera..."
+
+#: datacopydlg.c:296
+#, c-format
+msgid "%p%% done"
+msgstr "%p%% aconseguit"
+
+#: datacopydlg.c:326
+msgid "Stop"
+msgstr "Parar"
+
+#: datatrack.c:96
+#, c-format
+msgid ""
+"%s:\n"
+"Only the first data track of a CD can be used as a\n"
+"data source right now."
+msgstr ""
+"%s:\n"
+"Sols la primera pista de dades d'un CD pot ser emprada\n"
+"com a font de dades."
+
+#: datatrack.c:140
+#, c-format
+msgid "Datatrack #%2i of CD %#x"
+msgstr "Pista de dades  #%2i de CD %#x"
+
+#: datatrack.c:143
+#, c-format
+msgid "Datatrack #%02i of disc in drive %s"
+msgstr "Pista de dades #%02i del disc en la unitat %s"
+
+#: dependencies.c:43
+msgid "Debian/GNU Linux"
+msgstr "Debian/GNU Linux"
+
+#: dependencies.c:44
+msgid "The Debian Project"
+msgstr "The Debian Project"
+
+#: dependencies.c:45
+msgid "potato"
+msgstr "potato"
+
+#: dependencies.c:47
+msgid ""
+"The ultimate, free Linux distribution. Not too easy for beginners, though."
+msgstr "La distribuci lliure de linux. No molt fcil per a no iniciats, encara."
+
+#: dependencies.c:66
+msgid "MPEG123 MPEG Audio Player"
+msgstr "MPEG123 MPEG Audio Player"
+
+#: dependencies.c:67
+msgid "Michael Hipp"
+msgstr "Michael Hipp"
+
+#: dependencies.c:68
+msgid "0.59r (1999/Jun/15)"
+msgstr "0.59r (1999/Jun/15)"
+
+#: dependencies.c:72
+msgid ""
+"MPG123 is used by Gnometoaster to decode MP3 files for both playing and "
+"recording. Essentially mpg123 is a commandline mpeg audio player capable of "
+"handling all sorts of mpeg audio streams (MPEG1+2, Layers 1 through 3)."
+msgstr ""
+"Gnometoater empra MPG123 per a decodificar i escoltar arxius MP3.\n"
+"Bsicament mpg123 s un programa (mode text) per a poder escoltar \n"
+"varis tipus de audio mpeg (MPEG1+2, capes 1 a 3)."
+
+#: dependencies.c:94
+msgid "Sound Exchange"
+msgstr "Intercanvi de So"
+
+#: dependencies.c:95
+msgid "Lance Norskog"
+msgstr "Lance Norskog"
+
+#: dependencies.c:96
+msgid "12.16"
+msgstr "12.16"
+
+#: dependencies.c:100
+msgid ""
+"Gnometoaster is using sox to convert various types of .wav files into cd "
+"digital audio streams suitable for direct recording.\n"
+"SoX is a sound file format converter for Unix and DOS PCs written by Lance "
+"Norskog and other invaluable contributors. It also does sample rate "
+"conversion and some sound effects. It's the swiss army knife of sound tools: "
+"the interface isn't great, but it does almost everything. SoX uses file "
+"suffices to determine the nature of a sound sample file. If it finds the "
+"suffix in its list, it uses the appropriate read or write handler to deal "
+"with that file. SoX has an auto-detect feature that attempts to figure out "
+"the nature of an unmarked sound sample. It works very well. This is the "
+"'auto' file format. (Source: http://www.spies.com/Sox/)."
+msgstr ""
+
+#: dependencies.c:124
+msgid "mkisofs"
+msgstr "mkisofs"
+
+#: dependencies.c:125
+msgid "Eric Youngdale/Yggdrasil Computing Inc."
+msgstr "Eric Youngdale/Yggdrasil Computing Inc."
+
+#: dependencies.c:126
+msgid "1.13"
+msgstr "1.13"
+
+#: dependencies.c:130
+msgid ""
+"Gnometoaster is using 'mkisofs' to create ISO9660 filesystems.\n"
+"That's what you usually do when creating a data CD (CDROM) with\n"
+"files on it.\n"
+"mkisofs is effectively a pre-mastering program to generate the iso9660 "
+"filesystem - it takes a snapshot of a\n"
+"given directory tree, and generates a binary image which will correspond to "
+"an iso9660 filesystem when written to a\n"
+"block device.(Source:mkisofs manpage (man 8 mkisofs))\n"
+msgstr ""
+
+#: dependencies.c:153
+msgid "cdda2wav"
+msgstr "cdda2wav"
+
+#: dependencies.c:154
+msgid "Heiko Eissfeldt (heiko@colossus.escape.de)"
+msgstr "Heiko Eissfeldt (heiko@colossus.escape.de)"
+
+#: dependencies.c:155 dependencies.c:211
+msgid "1.9"
+msgstr "1.9"
+
+#: dependencies.c:159
+msgid ""
+"'cdda2wav' is used by Gnometoaster to extract Digital Audio from\n"
+"audio compact discs for either encoding or burning.\n"
+"cdda2wav can retrieve audio tracks from CDROM drives (see\n"
+"[cdda2wav-]README  for  a list of drives) that are capable of reading\n"
+"audio data digitally to the host (CDDA). (Source:man cdda2wav)\n"
+msgstr ""
+
+#: dependencies.c:175
+msgid "GoGo"
+msgstr "GoGo"
+
+#: dependencies.c:176
+msgid "PEN@MarineCat and shigeo"
+msgstr "PEN@MarineCat and shigeo"
+
+#: dependencies.c:177
+msgid "ver. 2.22 (Nov  8 1999)"
+msgstr "ver. 2.22 (Nov  8 1999)"
+
+#: dependencies.c:181
+msgid ""
+"Gnometoaster can use this program to encode audio tracks and .wav\n"
+"files into mp3 files.\n"
+"GoGo is an assembly optimized version of the free Lame encoder\n"
+"yielding about 6x encoding speed on my K6-3-400 machine.\n"
+"For many, Lame is considered the best free mp3 encoder available\n"
+"Note that in some countries it may be illegal to use this encoder\n"
+"because of software patent issues. See http://gnometoaster.rulez.org\n"
+"to find more informations about how you can protest against\n"
+"certain legal practices that may eventually make life a lot more\n"
+"difficult for us with intellectual property becoming more and more\n"
+"important an issue.\n"
+msgstr ""
+
+#: dependencies.c:209
+msgid "Cdrecord"
+msgstr "Cdrecord"
+
+#: dependencies.c:210
+msgid "Joerg Schilling (schilling@fokus.gmd.de)"
+msgstr "Joerg Schilling (schilling@fokus.gmd.de)"
+
+#: dependencies.c:215
+msgid ""
+"Gnometoaster depends on cdrecord for cd recording and CDRW blanking.\n"
+"Note that if cdrecord isn't installed, Gnometoaster may still be able\n"
+"to record discs in disc-at-once mode using cdrdao.\n"
+"Cdrecord is a commmandline cd recording program capable of using\n"
+"most cd recorders in track-at-once and some of them in disc-at-once mode\n"
+"(a rule of thumb would be to check wether your drive supports cuesheet "
+"recording).\n"
+"Please go to the cdrecord homepage to get a list of drives currently\n"
+"supported by this program. If your drive isn't supported, you can try\n"
+"using it with cdrdao, yet another backend Gnometoaster is supporting\n"
+msgstr ""
+
+#: dependencies.c:241
+msgid "CdrDAO"
+msgstr "CdrDAO"
+
+#: dependencies.c:242
+msgid "Andreas Mueller <mueller@daneb.ping.de>"
+msgstr "Andreas Mueller <mueller@daneb.ping.de>"
+
+#: dependencies.c:243
+msgid "1.1.4"
+msgstr "1.1.4"
+
+#: dependencies.c:247
+msgid ""
+"Gnometoaster can use cdrdao to record CDRs in disc-at-once mode.\n"
+"Please make sure your drive is supported by cdrdao before trying\n"
+"to record a disc with this backend. As you don't have cdrdao\n"
+"installed, you can either turn off the \"Disc-at-once\" switch\n"
+"in Gnometoaster's recording panel or uncheck the \"Use alternative\n"
+"DAO client\" option in Gnometoaster's recording options.\n"
+"Gnometoaster will then use cdrecord to write your CDs\n"
+msgstr ""
+
+#: dependencies.c:265
+msgid "Shorten Lossless Audio Compressor"
+msgstr ""
+
+#: dependencies.c:266
+msgid "Tony Robinson and SoftSound Ltd"
+msgstr "Tony Robinson and SoftSound Ltd"
+
+#: dependencies.c:267
+msgid "version 3.2"
+msgstr "version 3.2"
+
+#: dependencies.c:271
+msgid ""
+"Gnometoaster can use shorten to burn .shn files to a CDR on-the-fly.The "
+"program can also be used to encode audio into .shn files.Shorten is a "
+"fileformat used for lossless audio compression and is therefore especially "
+"useful for musicians (e.g. for exchanging demo tapes).See http://www.etree."
+"org for details."
+msgstr ""
+
+#: dependencies.c:281
+msgid "Shorten Lossless Audio Compression Tool"
+msgstr "Shorten Lossless Audio Compression Tool"
+
+#: dependencies.c:282
+msgid "Jason Jordan"
+msgstr "Jason Jordan"
+
+#: dependencies.c:283
+msgid "version 1.01"
+msgstr "version 1.01"
+
+#: dependencies.c:287
+msgid ""
+"GnomeToaster needs this program if you want to record shorten files to a "
+"CDR. It is used to calculate the exact space required by your shorten file "
+"on the medium.For informations about the Shorten format, visit http://www."
+"etree.org."
+msgstr ""
+
+#: dependencies.c:302
+msgid "Ogg-Vorbis commandline decoder "
+msgstr "Decompressor de Ogg-Vorbis en linea de comandes"
+
+#: dependencies.c:303
+msgid "Kenneth Arnold"
+msgstr "Kenneth Arnold"
+
+#: dependencies.c:304
+msgid "version 1.0beta4"
+msgstr "version 1.0beta4"
+
+#: dependencies.c:308
+msgid ""
+"GnomeToaster needs this program to decode ogg-vorbis files (.ogg).Ogg-Vorbis "
+"is a totally free implementation of a lossy audio-compression scheme similar "
+"to Fraunhofer's mp3. It yields slightly better quality at the same bitrate "
+"and is free of any patented alghorithms."
+msgstr ""
+
+#: dependencies.c:323
+msgid "Ogg Vorbis File Information viewer"
+msgstr "Ogg Vorbis File Information viewer"
+
+#: dependencies.c:324
+msgid "James Atwill"
+msgstr "James Atwill"
+
+#: dependencies.c:325
+msgid "Jan 2001"
+msgstr "Jan 2001"
+
+#: dependencies.c:329
+msgid ""
+"GnomeToaster needs this program to determine the playing time of an ogg-"
+"vorbis file. Because it is not currently part of the main vorbis-tools "
+"distribution, a packaged version of the source can be downloaded from http://"
+"gnometoaster.rulez.org"
+msgstr ""
+
+#: dependencies.c:445
+msgid "Dependency database"
+msgstr "Base de dades de dependncies"
+
+#: dependencies.c:460
+#, c-format
+msgid ""
+"Gnometoaster failed to run the following command\n"
+"on your system:\n"
+"%s\n"
+msgstr ""
+"Gnometoaster ha fallat en ejecutar la\n"
+"segent comanda en el teu sistema:\n"
+"%s\n"
+
+#: dialog.c:63 dialog.c:162 dialog.c:235 forms.c:148 menufunc.c:87
+#: dialog.c:215 dialog.c:239 menufunc.c:89
+msgid "OK"
+msgstr "OK"
+
+#: dialog.c:98 dialog.c:211 dialog.c:215
+msgid "YES"
+msgstr "YES"
+
+#: dialog.c:105 dialog.c:222 dialog.c:226
+msgid "NO"
+msgstr "NO"
+
+#: dialog.c:169 forms.c:157 preferences.c:414 preferences.c:432
+msgid "Cancel"
+msgstr "Cancellar"
+
+#: filelist.c:29 filelist.c:30
+msgid "Filename"
+msgstr "Nom fitxer"
+
+#: filelist.c:31 filelist.c:32
+msgid "Filesize"
+msgstr "Tamany fitxer"
+
+#: fileman.c:265
+msgid "Adding files to the filesystem"
+msgstr "Afegir arxius al sistema de fitxers"
+
+#: fileman.c:273
+msgid "(please wait)"
+msgstr "(Esperar)"
+
+#: fileman.c:377
+#, c-format
+msgid ""
+"GnomeToaster was unable to set tagging informations\n"
+"for the file it just encoded because the program responsible for doing this\n"
+"could not be found on your system:\n"
+"%s.\n"
+msgstr ""
+
+#: fileman.c:521
+msgid "Encoding File"
+msgstr "Comprimint Fitxer"
+
+#: fileman.c:563
+#, c-format
+msgid ""
+"No valid encoder could be found for the\n"
+"selected tracktype %s.\n"
+"Please create a corresponding entry in the\n"
+"encoder section of Gnometoaster's filetype\n"
+"registry.\n"
+msgstr ""
+"Compressor no valid per a el tipus de pista\n"
+"seleccionada %s.\n"
+"Crea una entrada en la secci compressor\n"
+"en la pestanya tipus de fitxer\n"
+
+#: fileman.c:637
+#, c-format
+msgid "Encode as %s"
+msgstr "Comrimeis com a %s"
+
+#: fileman.c:713 fsedit.c:348 fsedit.c:549
+msgid "Create Directory"
+msgstr "Crear directori"
+
+#: fileman.c:714
+msgid "New Directory"
+msgstr "Nou directori"
+
+#: fileman.c:774
+msgid "Rename File"
+msgstr "Reanomenar fitxer"
+
+#: filetypes.c:167
+msgid "Select File"
+msgstr "Seleccionar fitxer"
+
+#: filetypes.c:265
+#, c-format
+msgid "Default for Tracktype %s"
+msgstr "Tipus de pista predeterminat per a %s"
+
+#: filetypes.c:267
+msgid "Encoder settings"
+msgstr "Opcions del Compressor"
+
+#: filetypes.c:271
+msgid "Encoder call"
+msgstr "Crida al compressor"
+
+#: filetypes.c:275
+msgid "Encoder needs little Endian"
+msgstr "El compressor necessita little Endian"
+
+#: filetypes.c:283
+msgid "Set title info with"
+msgstr "Posat informaci de ttol amb"
+
+#: filetypes.c:318 filetypes.c:418
+msgid "default"
+msgstr "predeterminat"
+
+#: filetypes.c:342
+msgid "precache file before writing"
+msgstr ""
+
+#: filetypes.c:366 filetypes.c:384
+msgid "S"
+msgstr "S"
+
+#: filetypes.c:371 filetypes.c:389
+msgid "Select program"
+msgstr "Seleccionar programa"
+
+#: filetypes.c:402
+msgid "Encoding"
+msgstr "Comprimint"
+
+#: filetypes.c:903
+msgid "Filetypes"
+msgstr "Tipus de Fitxers"
+
+#: filetypes.c:911
+msgid "Suffix"
+msgstr "Sufixe"
+
+#: filetypes.c:917
+msgid "Tracktype"
+msgstr "Tipus de Pista"
+
+#: filetypes.c:923
+msgid "PC"
+msgstr "PC"
+
+#: filetypes.c:929
+msgid "Format"
+msgstr "Format"
+
+#: filetypes.c:935
+msgid "Filtername"
+msgstr "Nom del filtre"
+
+#: filetypes.c:941 filetypes.c:953
+msgid " "
+msgstr " "
+
+#: filetypes.c:947
+msgid "Detect Size"
+msgstr "Detectar Tamany"
+
+#: filetypes.c:959
+msgid "Add Filetype"
+msgstr "Afegir un nou tipus de fitxer"
+
+#: fillstate.c:104
+#, c-format
+msgid "Fillstate %u MB/ %u MB ( %3.0f %% , %u:%02u/%u:%02u Minutes )"
+msgstr "Estat del CD %u MB/ %u MB ( %3.0f %% , %u:%02u/%u:%02u Minuts )"
+
+#: fillstate.c:474
+msgid "Fillstate Setup"
+msgstr "Configuraci de la Barra de Estat"
+
+#: fillstate.c:477
+msgid "Use Themes"
+msgstr "Emprar Tema"
+
+#: fillstate.c:498
+msgid "Add Color"
+msgstr "Afegir Color"
+
+#: fillstate.c:512
+msgid "Select Track Color"
+msgstr "Selecciona Color de la Pista"
+
+#: forms.c:166 preferences.c:385 preferences.c:403
+msgid "Undo"
+msgstr "Desfer"
+
+#: fsedit.c:100
+#, c-format
+msgid ""
+"Error adding items to the Filesystem:\n"
+"%s\n"
+"Please check the result.\n"
+msgstr ""
+"Error al afegir:\n"
+"%s\n"
+"Comprova el resultat.\n"
+
+#: fsedit.c:107
+msgid "Unknown Filesystem error"
+msgstr "Error de sistema de fitxers desconegut"
+
+#: fsedit.c:347
+msgid "Delete File(s)"
+msgstr "Borrar Fitxer(s)"
+
+#: fsedit.c:347
+msgid "Delete marked file(s)/director(y/ies)"
+msgstr "Eliminar fitxer(s)/directori(s) marcats"
+
+#: fsedit.c:348
+msgid "Create a directory within the current file list"
+msgstr "Crear directori dins de la llista actual"
+
+#: fsedit.c:349
+msgid "Rename File(s)"
+msgstr "Reanomenad Fitxer(s)"
+
+#: fsedit.c:349
+msgid "Rename selected file(s)"
+msgstr "Reanomenar fitxer(s) seleccionats"
+
+#: fsedit.c:350
+msgid "Make Boot Image"
+msgstr "Crear imatge d'arranc"
+
+#: fsedit.c:350
+msgid "Make current cd bootable using this boot image"
+msgstr "Fer autoarrencable el CD actual amb aquesta imatge d'arranc"
+
+#: fsedit.c:351
+msgid "Convert to.."
+msgstr "Convertira a ..."
+
+#: fsedit.c:351
+msgid "Convert this file to another filetype"
+msgstr "Convertir aquest fitxer a un altre tipus"
+
+#: fsedit.c:520
+msgid "ISO Filesystem Volume ID"
+msgstr "Identificador del volum ISO"
+
+#: fsedit.c:548
+msgid "Delete Directory"
+msgstr "Eliminar Directori"
+
+#: fsedit.c:548
+msgid "Delete a directory"
+msgstr "Eliminar un directori"
+
+#: fsedit.c:549
+msgid "Create a directory within the currently selected dir"
+msgstr ""
+
+#: fsedit.c:550
+msgid "Rename VOLUME/Directory"
+msgstr "Reanomenar VOLUM/Directory"
+
+#: fsedit.c:550
+msgid "Rename a directory"
+msgstr "Reanomenar directori"
+
+#: helpings.c:230
+#, c-format
+msgid "%02i hour(s) %02i min. %02i sec."
+msgstr "%02i hora(es) %02i min. %02i seg."
+
+#: isotrack.c:231
+msgid "Gnometoaster-Filesystem (ISO)"
+msgstr "Pseudo sistema de fitxers de Gnometoaster (ISO)"
+
+#: isotrack.c:234
+msgid "Gnome Toaster"
+msgstr "Gnome Toaster"
+
+#: isotrack.c:294
+msgid "Create ISO Track with"
+msgstr "Crear pista ISO amb"
+
+#: isotrack.c:299
+msgid "Multisession Options"
+msgstr "Opcions de multissessi"
+
+#: isotrack.c:304
+msgid "Bootable Options"
+msgstr "Opcions del Arrencable"
+
+#: isotrack.c:309
+msgid "Use create call to determine tracksize"
+msgstr ""
+
+#: isotrack.c:319
+msgid "Determine tracksize option"
+msgstr "Determinar tamany de la pista"
+
+#: isotrack.c:324
+msgid "Determine tracksize with"
+msgstr "Determinar tamany de la pista amb"
+
+#: isotrack.c:329
+msgid "Calculate tracksize as"
+msgstr "Calcular tamany pista com a"
+
+#: isotrack.c:334
+msgid "Mount an iso image with"
+msgstr "Montar una imatge ISO amb"
+
+#: isotrack.c:339
+msgid "Unmount an iso image with"
+msgstr "Desmontar un imatge iso amb"
+
+#: isotrack.c:344
+msgid "Precache before writing"
+msgstr ""
+
+#: isotrack.c:354
+msgid "ISO Temporary files"
+msgstr "Arxius temporals ISO"
+
+#: menubar.c:27
+msgid "Import session"
+msgstr "Importar Sessi"
+
+#: menubar.c:28 menubar.c:36
+msgid "Import the last session of this cd"
+msgstr "Importar la ultima sessi d'aquest CD"
+
+#: menubar.c:31
+msgid "Delete Filesystem"
+msgstr "Borrar el Sistema de Fitxers"
+
+#: menubar.c:32 menubar.c:40
+msgid "Delete the contents of fsedit"
+msgstr "Borrar els continguts de fsedit"
+
+#: menubar.c:41
+msgid "Refresh Tools list"
+msgstr "Refrescar la llista d'utilitats"
+
+#: menubar.c:42 menubar.c:50
+msgid "scan for new plugins"
+msgstr "Examinar per a nous endollats(plugins)"
+
+#: menubar.c:58
+msgid "Tools"
+msgstr "Utilitats"
+
+#: menufunc.c:72 menufunc.c:74
+msgid "About Gnometoaster"
+msgstr "A propsit de Gnometoaster"
+
+#: precachetrack.c:201
+msgid " (precached)"
+msgstr ""
+
+#: precachetrack.c:216
+msgid "precaching file..."
+msgstr ""
+
+#: preferences.c:162 preferences.c:167
+#, c-format
+msgid ""
+"Gnometoaster failed to create it's symbolic link area\n"
+"which is essential for filesystem creation. Please make\n"
+"sure that the user running Gnometoaster has permission\n"
+"to create the following directory:\n"
+"%s\n"
+msgstr ""
+
+#: preferences.c:373 preferences.c:391
+msgid "GnomeToaster Preferences"
+msgstr "Preferncies de GnomeToaster"
+
+#: preferences.c:394 preferences.c:412
+msgid "Apply"
+msgstr "Aplicar"
+
+#: preferences.c:424 preferences.c:442
+msgid "Common"
+msgstr "Com"
+
+#: preferences.c:429 preferences.c:447
+msgid "Temp Files"
+msgstr "Arxius Temporals"
+
+#: preferences.c:434 preferences.c:452
+msgid "Precache Data Tracks:"
+msgstr ""
+
+#: preferences.c:440 preferences.c:458
+msgid "Use GNOME compatible drag and drop scheme"
+msgstr "Emprar sistema d'arrossegar i soltar compatible amb GNOME"
+
+#: preview.c:120
+#, c-format
+msgid ""
+"The selected audio driver couldn't be initialized\n"
+"Please make sure your sound hardware is configured\n"
+"properly, all drivers have been loaded and access permissions have\n"
+"been set properly.\n"
+"The error message as received from the Operating System was:\n"
+"'%s'\n"
+msgstr ""
+
+#: preview.c:294
+msgid "Drag files/tracks to be previewed here"
+msgstr "Arrossegar ac els arxius/pistes per a una vista preliminar"
+
+#: record.c:67
+msgid ""
+"\n"
+"Child exited unexpectedly."
+msgstr ""
+
+#: record.c:150
+msgid "CD recording process finished."
+msgstr "Procs de Gravaci del CD finalitzat."
+
+#: record.c:242
+msgid ""
+"\n"
+"fixating CDROM."
+msgstr ""
+"\n"
+"fixant el CDROM."
+
+#: record.c:279
+msgid "fixating cdrom"
+msgstr "fixant el cdrom"
+
+#: record.c:351
+#, c-format
+msgid "Track %i/%i"
+msgstr "Pista %i/%i"
+
+#: record.c:641
+msgid "Unnamed disc"
+msgstr "Disc sense ttol"
+
+#: record.c:642
+msgid "No Artist"
+msgstr "Sense Autor"
+
+#. done writing toc file
+#: record.c:687
+#, c-format
+msgid "Recording %i bytes to CD"
+msgstr "Gravant %i bytes al CD"
+
+#: record.c:704
+msgid "Writing CD"
+msgstr "Gravant CD"
+
+#: record.c:714
+msgid "CD status"
+msgstr "Estat del CD"
+
+#: record.c:715
+msgid "Please wait"
+msgstr "Espereu"
+
+#: record.c:727
+msgid "Track Status"
+msgstr "Estat de la pista"
+
+#: record.c:763
+msgid "No recordable Tracks found."
+msgstr "Cap pista per a gravar trobada."
+
+#: record.c:767
+msgid "One or more tracks are unable to deliver their content."
+msgstr "Una o ms pistes no han sigut capa de mostrar el seu contingut."
+
+#: record.c:776
+msgid ""
+"Preview Player active,\n"
+"recording process cancelled."
+msgstr ""
+"El reproductor est actiu,\n"
+"gravaci cancellada."
+
+#: record.c:811
+msgid "CDRW blanking complete."
+msgstr "Borrat del CDRW aconseguit."
+
+#: record.c:857
+msgid "blanking CDRW."
+msgstr "Borrant CDRW."
+
+#: record.c:876
+msgid "Blanking CD/RW"
+msgstr "Borrant CD/RW"
+
+#. General recording options
+#: record.c:1029
+msgid "General"
+msgstr "General"
+
+#: record.c:1032
+msgid "Recorder Speed"
+msgstr "Velocitat del gravador"
+
+#: record.c:1043
+msgid "Eject disc"
+msgstr "Expulsar disc"
+
+#: record.c:1054
+msgid "Reload disc"
+msgstr "Recarregar el disc"
+
+#: record.c:1065
+msgid "Dummy Write"
+msgstr "Gravaci de prova"
+
+#: record.c:1080
+msgid "Disc at once"
+msgstr "Disc at once"
+
+#: record.c:1114
+msgid "CD-Text"
+msgstr "CD-Text"
+
+#. recording frame
+#: record.c:1151
+msgid "Recording"
+msgstr "Gravant"
+
+#: record.c:1155 toolbar.c:30 toolbar.c:31
+msgid "Record"
+msgstr "Gravar"
+
+#: record.c:1161
+msgid "Multisession"
+msgstr "Multissessi"
+
+#: record.c:1172
+msgid "Buffer-Underrun Protection"
+msgstr "Protecci Buffer-Underrun"
+
+#: record.c:1192
+msgid "Fixate"
+msgstr "Fixar"
+
+#: record.c:1224 toolbar.c:27 toolbar.c:28
+msgid "Clear Disc"
+msgstr "Borrar Disc"
+
+#: record.c:1230
+msgid "TOC only"
+msgstr "Sols TOC"
+
+#. common iso options
+#: record.c:1251
+msgid "ISO Options"
+msgstr "Opcions ISO"
+
+#: record.c:1254
+msgid "Boot Image"
+msgstr "Imatge d'arranc"
+
+#: record.c:1270
+msgid "Make Bootable"
+msgstr "Crear autoarrencable"
+
+#: record.c:1300
+msgid "Recorder"
+msgstr "Gravadora"
+
+#: record.c:1305
+msgid "recording client"
+msgstr "Programa de gravaci"
+
+#: record.c:1309
+msgid "Use single call"
+msgstr "Emprar cridada nica"
+
+#: record.c:1317
+msgid "Single call FIFO"
+msgstr "Crida al FIFO nica"
+
+#: record.c:1323
+msgid "Write whole CD with"
+msgstr "Gravar el CD senser amb"
+
+#: record.c:1327
+msgid "Track Item"
+msgstr ""
+
+#: record.c:1334
+msgid "Fixating command"
+msgstr "parametres per a fixar"
+
+#: record.c:1339
+msgid "Trackwrite command"
+msgstr "Parmetres per escritura de pistes"
+
+#: record.c:1344
+msgid "Blank disc command"
+msgstr "Parmetres per borrar"
+
+#: record.c:1349
+msgid "Eject disc command"
+msgstr "Parametres per a expulsar"
+
+#: record.c:1354
+msgid "Reload Disc with"
+msgstr "Recarregar Disc amb"
+
+#: record.c:1359
+msgid "Multisession Info"
+msgstr "Informaci de la multissesi"
+
+#: record.c:1365
+msgid "DAO Recording Client"
+msgstr "Programa per gravar DAO"
+
+#: record.c:1369
+msgid "Use alternative client"
+msgstr "Emprar programa alternatiu"
+
+#: record.c:1377
+msgid "DAO Write Command"
+msgstr "Parametres d'escritura e DAO"
+
+#: record.c:1382 record.c:1392
+msgid "Scsi drive inquiry"
+msgstr "Preguntar SCSI dispossitiu"
+
+#: record.c:1387 record.c:1397
+msgid "Get drive type"
+msgstr "Agafar tipus de dispossitiu"
+
+#: record.c:1392 record.c:1402
+msgid "Get vendor name"
+msgstr "Agafar nom del venedor"
+
+#: record.c:1397 record.c:1407
+msgid "Get drive identification"
+msgstr "Agafar identificaci del dispossitiu"
+
+#: record.c:1402 record.c:1412
+msgid "Get cdrom type"
+msgstr "Agafar el tipus de CDROM"
+
+#: record.c:1407 record.c:1417
+msgid "Mountpoint lookup file"
+msgstr "Arxiu de punts de montatge"
+
+#: record.c:1412 record.c:1422
+msgid "Get mountpoint with"
+msgstr "Agafar punt de montatge amb"
+
+#: rterm.c:58
+msgid "Client output"
+msgstr "Eixida del programa"
+
+#: rterm.c:77
+msgid "GnomeToaster Recording Terminal"
+msgstr "Terminal de Gravaci del GnomeToaster"
+
+#: selectlist.c:738 selectlist.c:739
+msgid "Select destination directory"
+msgstr "Seleccionar dest de la gravaci"
+
+#: source.c:97 source.c:98
+msgid "Source Area"
+msgstr ""
+
+#. Main File selection Tree
+#: source.c:108 source.c:109
+msgid "Unix Tree"
+msgstr "Arbre Unix"
+
+#: source.c:127 source.c:128
+msgid "Internal structures"
+msgstr "Estructures Internes"
+
+#. CDROMs as source for track streams
+#: source.c:136 source.c:137
+msgid "CDROM drives"
+msgstr "Dispossitius CDROM"
+
+#: stdfiletrack.c:267
+#, c-format
+msgid ""
+"The tool necessary to detect the tracksize of\n"
+"files with the suffix '%s' couldn't be found on\n"
+"your system. In it's current configuration,\n"
+"GnomeToaster is trying to execute the following command\n"
+"to obtain the tracksize:\n"
+"%s\n"
+msgstr ""
+
+#: stdfiletrack.c:441
+#, c-format
+msgid ""
+"The Filter program for the \n"
+"suffix '%s' couldn't be found on your system.\n"
+"Please make sure your filetype registry\n"
+"is setup properly.\n"
+"Your current configuration is calling the following\n"
+"command to create the desired track:\n"
+"%s\n"
+msgstr ""
+
+#: toolbar.c:15 toolbar.c:16
+msgid "Quit"
+msgstr "Eixir"
+
+#: toolbar.c:15 toolbar.c:16
+msgid "Quit Application"
+msgstr "Eixir de l'aplicaci"
+
+#: toolbar.c:19 toolbar.c:20
+msgid "Preferences..."
+msgstr "Preferncies..."
+
+#: toolbar.c:19 toolbar.c:20
+msgid "Set Preferences"
+msgstr "Establir preferncies"
+
+#: toolbar.c:23 toolbar.c:24
+msgid "Import"
+msgstr "Importar"
+
+#: toolbar.c:23 toolbar.c:24
+msgid "Import the last session"
+msgstr "Importar la ltima sessi"
+
+#: toolbar.c:27 toolbar.c:28
+msgid "Clear a CD-RW"
+msgstr "Borrar un CD-RW"
+
+#: toolbar.c:30 toolbar.c:31
+msgid "Start recording"
+msgstr "Comenar la gravaci"
+
+#: trackedit.c:39
+msgid "Delete Track"
+msgstr "Borrar Pista"
+
+#: trackedit.c:39
+msgid "Remove selected track(s) from the tracklist"
+msgstr "Eliminar les pistes seleccionades"
+
+#: trackedit.c:40
+msgid "Encode as.."
+msgstr "Codificar com a .."
+
+#: trackedit.c:40
+msgid "Encode this object to a certain filetype"
+msgstr ""
+
+#: trackedit.c:41
+msgid "Rename Track.."
+msgstr "Reanomenar Pista.."
+
+#: trackedit.c:41
+msgid "Change Title, Artist and Name of a track"
+msgstr "Canviar Ttol, Autor i Nom de la Pista"
+
+#: trackedit.c:58
+msgid "Rename track"
+msgstr "Reanomenar pista"
+
+#: trackedit.c:62
+msgid "Track Name"
+msgstr "Nom de la pista"
+
+#: trackedit.c:66
+msgid "Title"
+msgstr "Ttol"
+
+#: trackedit.c:70
+msgid "Artist"
+msgstr "Autor"
+
+#: trackedit.c:388
+msgid ""
+"You have used the Filesystem Editor but\n"
+"the ISO track is currently not in the list of tracks\n"
+"to be burned. Do you want me to add it for you ?\n"
+msgstr ""
+
+#: trackedit.c:397
+msgid "Unnamed disc (Click title to set disc info)"
+msgstr "Disc sense ttol (Polsa ttol per a introdur el ttol)"
+
+#: trackedit.c:438
+msgid "Set disc informations"
+msgstr "Introdur informaci del disc"
+
+#: trackedit.c:439
+msgid "Disc Title"
+msgstr "Ttol del disc"
+
+#: trackedit.c:440
+msgid "Artist/Author"
+msgstr "Artista/Autor"
+
+#: tracklist.c:138
+#, c-format
+msgid "%s by %s"
+msgstr "%s per %s"
+
+#: tracklist.c:139
+#, c-format
+msgid "Medium by %s"
+msgstr ""
+
+#: tracklist.c:140
+#, c-format
+msgid "%s"
+msgstr "%s"
+
+#: tracklist.c:167
+msgid "Trackname"
+msgstr "Nom de la Pista"
+
+#: tracklist.c:202
+msgid "Tracksize"
+msgstr "Tamany de la pista"
+
+#: vfs.c:519
+msgid "Couldn't get sourcefile informations"
+msgstr "No s'ha pogut agafar certes informacions del fitxer font"
+
+#: vfs.c:609
+msgid "Cannot copy/move directory into itself"
+msgstr "No es pot copiar/moure un directori a ell mateix"
+
+#: vfs.c:761
+msgid "Couldn't get file informations"
+msgstr ""
+
+#: vfs_unixtree.c:360
+msgid "Can't add Symbolic link to a Symlinktree Filesystem"
+msgstr ""
+
+#. report error
+#: vfs_unixtree.c:409
+msgid "Read/Write error copying file"
+msgstr "Error de lectura/escritura al copiar arxius"
+
+#: cddrives.c:1101
+msgid "Choose recorder"
+msgstr ""
+
+#: configupdate.c:574
+msgid ""
+"This version of GnomeToaster will fix the recorder detection\n"
+"scheme to be able to properly autodetect some USB recorders.\n"
+"Thanks to Alec Flett for telling me about the problem and also\n"
+"providing a solution to it.\n"
+msgstr ""
+
+#: dialog.c:226
+msgid "CANCEL"
+msgstr ""
+
+#: menubar.c:17
+msgid "Home directory"
+msgstr ""
+
+#: menubar.c:18 toolbar.c:35
+msgid "Go to home directory"
+msgstr ""
+
+#: menubar.c:21
+msgid "Change _Recorder"
+msgstr ""
+
+#: menubar.c:22
+msgid "Change the currently used recorder"
+msgstr ""
+
+#: menubar.c:35
+msgid "_Import session"
+msgstr ""
+
+#: menubar.c:39
+msgid "_Delete Filesystem"
+msgstr ""
+
+#: menubar.c:49
+msgid "_Refresh Tools list"
+msgstr ""
+
+#: menubar.c:66
+msgid "_Tools"
+msgstr ""
+
+#: menusys.h:65
+msgid "About"
+msgstr ""
+
+#: menusys.h:65
+msgid "Program informations"
+msgstr ""
+
+#: menusys.h:66
+msgid "Exit"
+msgstr ""
+
+#: menusys.h:66
+msgid "Exit program"
+msgstr ""
+
+#: menusys.h:67
+msgid "Cut"
+msgstr ""
+
+#: menusys.h:67
+msgid "Cut selection"
+msgstr ""
+
+#: menusys.h:68
+msgid "Copy"
+msgstr ""
+
+#: menusys.h:68
+msgid "Copy selection"
+msgstr ""
+
+#: menusys.h:69
+msgid "Paste"
+msgstr ""
+
+#: menusys.h:69
+msgid "Paste selection"
+msgstr ""
+
+#: menusys.h:70
+msgid "Preferences"
+msgstr ""
+
+#: menusys.h:70
+msgid "Edit Preferences"
+msgstr ""
+
+#: menusys.h:71
+msgid "File"
+msgstr ""
+
+#: menusys.h:71
+msgid "File menu"
+msgstr ""
+
+#: menusys.h:72
+msgid "Edit menu"
+msgstr ""
+
+#: menusys.h:73
+msgid "Help"
+msgstr ""
+
+#: menusys.h:73
+msgid "Help menu"
+msgstr ""
+
+#: preferences.c:464
+msgid "Go to home directory at startup"
+msgstr ""
+
+#: record.c:222
+msgid "GnomeToaster is calling the following command:"
+msgstr ""
+
+#: record.c:1382
+msgid "Scsi bus scan command"
+msgstr ""
+
+#: record.c:1387
+msgid "Scsi parse scan output"
+msgstr ""
+
+#: toolbar.c:35
+msgid "Home"
+msgstr "Inicio"
diff -urN gtoaster.orig/po/de.po gtoaster/po/de.po
--- gtoaster.orig/po/de.po	Sun Jun  8 20:13:20 2003
+++ gtoaster/po/de.po	Sun Jun  8 20:38:35 2003
@@ -6,13 +6,13 @@
 #, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
+"Project-Id-Version: 1.0Beta6\n"
 "POT-Creation-Date: 2002-08-28 08:50+0200\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
+"PO-Revision-Date: 2002-08-28 08:50+0200\n"
+"Last-Translator: <unknown@not-assigned.org>\n"
+"Language-Team: Deutch\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=CHARSET\n"
+"Content-Type: text/plain; charset=iso-8859-1\n"
 "Content-Transfer-Encoding: 8bit\n"
 
 #: dialog.c:90 dialog.c:91 dialog.c:96 dialog.c:98 dialog.c:204 dialog.c:211
diff -urN gtoaster.orig/po/fi.po gtoaster/po/fi.po
--- gtoaster.orig/po/fi.po	Sun Jun  8 20:13:20 2003
+++ gtoaster/po/fi.po	Sun Jun  8 21:13:02 2003
@@ -6,13 +6,13 @@
 #, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
+"Project-Id-Version: 1.0Beta6\n"
 "POT-Creation-Date: 2002-08-28 08:50+0200\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
+"PO-Revision-Date: 2002-08-28 08:50+0200\n"
+"Last-Translator: Unknown <unknown@not-assigned.org>\n"
+"Language-Team: Finnish\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=CHARSET\n"
+"Content-Type: text/plain; charset=iso-8859-1\n"
 "Content-Transfer-Encoding: 8bit\n"
 
 #: dialog.c:90 dialog.c:91 dialog.c:96 dialog.c:98 dialog.c:204 dialog.c:211
diff -urN gtoaster.orig/po/fr.po gtoaster/po/fr.po
--- gtoaster.orig/po/fr.po	Sun Jun  8 20:13:20 2003
+++ gtoaster/po/fr.po	Sun Jun  8 20:47:52 2003
@@ -6,13 +6,13 @@
 #, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
+"Project-Id-Version: 1.0Beta6\n"
 "POT-Creation-Date: 2002-08-28 08:50+0200\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
+"PO-Revision-Date: 2002-08-28 08:50+2002\n"
+"Last-Translator: Unknown <unknown@not-assigned.org>\n"
+"Language-Team: French\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=CHARSET\n"
+"Content-Type: text/plain; charset=iso-8859-1\n"
 "Content-Transfer-Encoding: 8bit\n"
 
 #: audio.c:46
diff -urN gtoaster.orig/po/hu.po gtoaster/po/hu.po
--- gtoaster.orig/po/hu.po	Sun Jun  8 20:13:20 2003
+++ gtoaster/po/hu.po	Sun Jun  8 21:18:13 2003
@@ -6,13 +6,13 @@
 #, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
+"Project-Id-Version: 1.0Beta6\n"
 "POT-Creation-Date: 2002-08-28 08:50+0200\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
+"PO-Revision-Date: 2002-08-28 08:50+0200\n"
+"Last-Translator: Unknown <unknown@not-assigned.org>\n"
+"Language-Team: Hungarian\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=CHARSET\n"
+"Content-Type: text/plain; charset=iso-8859-2\n"
 "Content-Transfer-Encoding: 8bit\n"
 
 #: audio.c:46
diff -urN gtoaster.orig/po/ja.po gtoaster/po/ja.po
--- gtoaster.orig/po/ja.po	Sun Jun  8 20:13:20 2003
+++ gtoaster/po/ja.po	Sun Jun  8 20:37:42 2003
@@ -6,13 +6,13 @@
 #, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
+"Project-Id-Version: 1.0Beta6\n"
 "POT-Creation-Date: 2002-08-28 08:50+0200\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
+"PO-Revision-Date: 2002-08-28 08:50+0200\n"
+"Last-Translator: unknown <unknown@unknown.org>\n"
+"Language-Team: Japanese\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=CHARSET\n"
+"Content-Type: text/plain; charset=EUC-JP\n"
 "Content-Transfer-Encoding: 8bit\n"
 
 #: audio.c:46
diff -urN gtoaster.orig/po/lt.po gtoaster/po/lt.po
--- gtoaster.orig/po/lt.po	Sun Jun  8 20:13:20 2003
+++ gtoaster/po/lt.po	Sun Jun  8 21:28:46 2003
@@ -6,13 +6,13 @@
 #, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
+"Project-Id-Version: 1.0Beta6\n"
 "POT-Creation-Date: 2002-08-28 08:50+0200\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
+"PO-Revision-Date: 2002-08-28 08:50+0200\n"
+"Last-Translator: Unknown <unknown@not-assigned.org>\n"
+"Language-Team: Lithuanian\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=CHARSET\n"
+"Content-Type: text/plain; charset=iso-8859-13\n"
 "Content-Transfer-Encoding: 8bit\n"
 
 #: audio.c:46
diff -urN gtoaster.orig/po/no.po gtoaster/po/no.po
--- gtoaster.orig/po/no.po	Sun Jun  8 20:13:20 2003
+++ gtoaster/po/no.po	Sun Jun  8 20:45:06 2003
@@ -6,13 +6,13 @@
 #, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
+"Project-Id-Version: 1.0Beta6\n"
 "POT-Creation-Date: 2002-08-28 08:50+0200\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
+"PO-Revision-Date: 2002-08-28 08:50+0200\n"
+"Last-Translator: Unknown <unknown@not-assigned.org>\n"
+"Language-Team: Unknown\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=CHARSET\n"
+"Content-Type: text/plain; charset=iso-8859-1\n"
 "Content-Transfer-Encoding: 8bit\n"
 
 #: dialog.c:90 dialog.c:91 dialog.c:96 dialog.c:98 dialog.c:204 dialog.c:211
diff -urN gtoaster.orig/po/ro.po gtoaster/po/ro.po
--- gtoaster.orig/po/ro.po	Sun Jun  8 20:13:20 2003
+++ gtoaster/po/ro.po	Sun Jun  8 21:15:45 2003
@@ -6,13 +6,13 @@
 #, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
+"Project-Id-Version: 1.0Beta6\n"
 "POT-Creation-Date: 2002-08-28 08:50+0200\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
+"PO-Revision-Date: 2002-08-28 08:50+0200\n"
+"Last-Translator: Unknown <unknown@not-assigned.org>\n"
+"Language-Team: Romanian\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=CHARSET\n"
+"Content-Type: text/plain; charset=iso-8859-2\n"
 "Content-Transfer-Encoding: 8bit\n"
 
 #: audio.c:46
diff -urN gtoaster.orig/po/ru.po gtoaster/po/ru.po
--- gtoaster.orig/po/ru.po	Sun Jun  8 20:13:20 2003
+++ gtoaster/po/ru.po	Sun Jun  8 21:32:09 2003
@@ -6,13 +6,13 @@
 #, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
+"Project-Id-Version: 1.0Beta6\n"
 "POT-Creation-Date: 2002-08-28 08:50+0200\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
+"PO-Revision-Date: 2002-08-28 08:50+0200\n"
+"Last-Translator: Unknown <unknown@not-assigned.org>\n"
+"Language-Team: Russian\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=CHARSET\n"
+"Content-Type: text/plain; charset=koi8-r\n"
 "Content-Transfer-Encoding: 8bit\n"
 
 #: audio.c:46
diff -urN gtoaster.orig/po/ru.po~ gtoaster/po/ru.po~
--- gtoaster.orig/po/ru.po~	Thu Jan  1 01:00:00 1970
+++ gtoaster/po/ru.po~	Wed Aug 28 08:50:20 2002
@@ -0,0 +1,1897 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"POT-Creation-Date: 2002-08-28 08:50+0200\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=CHARSET\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: audio.c:46
+msgid "Audio System"
+msgstr " "
+
+#: audio.c:50
+msgid "Audio driver"
+msgstr " "
+
+#: audiotrack.c:247
+#, c-format
+msgid "CD %#x Audiotrack %02d"
+msgstr "CD %#x   %02d"
+
+#: audiotrack.c:251
+#, c-format
+msgid "CD in %s Audiotrack %02d"
+msgstr "CD  %s   %02d"
+
+#: audiotrack.c:302
+msgid "CD Digital Audio"
+msgstr " CD "
+
+#: audiotrack.c:306
+msgid "Audio track Client:"
+msgstr "  :"
+
+#: audiotrack.c:311
+msgid "Audio track reading:"
+msgstr "  :"
+
+#: audiotrack.c:316
+msgid "CD Text retrieval:"
+msgstr " CD :"
+
+#: audiotrack.c:320
+msgid "CD Text parse Title:"
+msgstr "   CD :"
+
+#: audiotrack.c:324
+msgid "CD Text parse Artist:"
+msgstr "   CD :"
+
+#: audiotrack.c:328
+msgid "CD Text parse Disc Title:"
+msgstr "    CD :"
+
+#: audiotrack.c:332
+msgid "CD Text parse Disc Artist:"
+msgstr "    CD :"
+
+#: audiotrack.c:337
+msgid "Precache Audio Tracks:"
+msgstr "-  :"
+
+#: cddb.c:892
+msgid "CDDB server"
+msgstr " CDDB"
+
+#: cddrives.c:160
+msgid "Drive Type"
+msgstr " "
+
+#: cddrives.c:161
+msgid "Manufacturer Name"
+msgstr " "
+
+#: cddrives.c:162
+msgid "Device Name"
+msgstr " "
+
+#: cddrives.c:163 cddrives.c:436 cddrives.c:491
+msgid "Scsi ID"
+msgstr "Scsi ID"
+
+#: cddrives.c:164 cddrives.c:440 cddrives.c:495
+msgid "Mountpoint"
+msgstr " "
+
+#: cddrives.c:422 cddrives.c:477
+msgid "Please enter drive data"
+msgstr ",   "
+
+#: cddrives.c:426 cddrives.c:481 cddrives.c:1074
+msgid "Model"
+msgstr ""
+
+#: cddrives.c:429 cddrives.c:484 cddrives.c:1075
+msgid "Manufacturer"
+msgstr ""
+
+#: cddrives.c:433 cddrives.c:488
+msgid "Device File"
+msgstr " "
+
+#: cddrives.c:444 cddrives.c:499
+msgid "This Drive is a CD writer"
+msgstr "    CD "
+
+#: cddrives.c:448 cddrives.c:503
+msgid "Use SCSI Interface for DAE"
+msgstr "  SCSI  DAE"
+
+#: cddrives.c:458 cdsource.c:119 cddrives.c:513
+msgid "(Model)"
+msgstr "()"
+
+#: cddrives.c:459 cddrives.c:514
+msgid "(Manufacturer)"
+msgstr "()"
+
+#: cddrives.c:460 cddrives.c:515
+msgid "/dev/scd0"
+msgstr "/dev/scd0"
+
+#: cddrives.c:461 cddrives.c:516
+msgid "0,0"
+msgstr "0,0"
+
+#: cddrives.c:462 cddrives.c:517
+msgid "/mnt/cdrom"
+msgstr "/mnt/cdrom"
+
+#: cddrives.c:556 cddrives.c:611
+msgid "CDROM and Recorder Setup"
+msgstr " CDROM  "
+
+#: cddrives.c:578 cddrives.c:633
+msgid "Add"
+msgstr ""
+
+#: cddrives.c:583 filetypes.c:412 cddrives.c:638
+msgid "Remove"
+msgstr ""
+
+#: cddrives.c:588 cddrives.c:643 menusys.h:72
+msgid "Edit"
+msgstr ""
+
+#: cddrives.c:593 cddrives.c:648
+msgid "Scan Bus"
+msgstr " "
+
+#: cddrives.c:601 cddrives.c:656
+msgid "Perform bus scan at startup"
+msgstr "    "
+
+#: configupdate.c:253 configupdate.c:266
+msgid ""
+"This Version of Gnometoaster is providing a new\n"
+"drive setup scheme. You have to reconfigure your\n"
+"CDROM and recorder settings. Usually, all it takes\n"
+"is pressing the \"Scan Bus\" button in Gnometoaster's\n"
+"new drive setup\n"
+msgstr ""
+"     \n"
+"  .   \n"
+" CDROM  . ,  \n"
+"   \" \"  \n"
+"  .\n"
+
+#: configupdate.c:443 configupdate.c:456
+msgid ""
+"A Configfile for a newer version\n"
+"of Gnometoaster was found.\n"
+"Continuing may be dangerous because\n"
+"of possible changes in the file\n"
+"structure which may destroy your\n"
+"entire configuration.\n"
+"Do you want to run Gnometoaster anyway ?\n"
+msgstr ""
+"    \n"
+"   . \n"
+"    - \n"
+"   ,  \n"
+"   .\n"
+"     ?\n"
+
+#: configupdate.c:451 configupdate.c:464
+msgid ""
+"Introducing multisession toc and dummy write\n"
+"functionality into gnometoaster requires a few\n"
+"modifications in gnometoaster's configuration setup\n"
+"I can do the necessary modifications\n"
+"automatically.You will lose\n"
+"your customized configuration\n"
+"of \"Preferences/Recorder/Trackwrite command\",\n"
+"\"Preferences/Recorder/Fixating Command\"\n"
+"and \"Preferences/Recorder/Clear Disc\"\n"
+"If you want to modify those entries yourself,\n"
+"just answer NO to this question.\n"
+"(New Variables rec_multisession,rec_dummymode)\n"
+msgstr ""
+"     \n"
+"     \n"
+"     \n"
+" .\n"
+"    \n"
+".    \n"
+"  \n"
+"\"//  \",\n"
+"\"// \" \n"
+" \"//  \".\n"
+"     \n"
+",      .\n"
+"(  rec_multisession,rec_dummymode)\n"
+
+#: configupdate.c:464 configupdate.c:477
+msgid ""
+"This version of Gnometoaster switches the underlying\n"
+"filesystem creator from mkhybrid to mkisofs.\n"
+"You can,of course,continue to use mkhybrid.\n"
+"Just answer no to this question in this case.\n"
+msgstr ""
+"    \n"
+"   \n"
+" mkhybrid  mkisofs. ,  \n"
+"  mkhybrid.\n"
+"  ,   \n"
+"  .\n"
+
+#: configupdate.c:469 configupdate.c:482
+msgid ""
+"This update fixes a bug in the mp3 client call\n"
+"which made gnometoaster record non-cd-quality\n"
+"mp3 files with 2x or 4x the speed of the original.\n"
+"(modifies filetype registry's mp3 entry).\n"
+"\n"
+"Also,El Torito bootable support requires your mkisofs\n"
+"calls to be modified.\n"
+msgstr ""
+"   \n"
+"  mp3 , - \n"
+"  mp3  \n"
+"-CD-   2x  4x\n"
+"   ( \n"
+"    mp3).\n"
+"\n"
+",   El Torito\n"
+"    mkisofs.\n"
+
+#: configupdate.c:476 configupdate.c:489
+msgid ""
+"This version of Gnometoaster enables setting the\n"
+"volume id of an iso track from fsedit.\n"
+"To accomplish this,the default mkisofs call\n"
+"in both the standard and the multisession version.\n"
+"has to be modified.\n"
+msgstr ""
+"     \n"
+"   ISO  fsedit.\n"
+"  ,  \n"
+" mkisofs  ,   ,\n"
+"    .\n"
+
+#: configupdate.c:482 configupdate.c:495
+msgid ""
+"This version of Gnometoaster makes it possible to calculate\n"
+"the tracksize of an isotrack accurately by calling mkisofs\n"
+"for that purpose. Some further cleanups rid the configuration\n"
+"of a separate multisession isotrack call,incorporating multi-\n"
+"session support into the main iso image call as an option.\n"
+"One of the immediate results of those efforts is that now\n"
+"we can safely give the calculated tracksize to cdrecord\n"
+"to support recorders needing this value in advance.\n"
+"However,to accomplish that,the cdrecord write_track call\n"
+"has to be updated.\n"
+msgstr ""
+"     \n"
+"  ISO    mkisofs  \n"
+".    \n"
+"     ISO,\n"
+"     \n"
+"  ISO  .\n"
+"     \n"
+" ,       \n"
+"   cdrecord   ,\n"
+"   . \n"
+",     \n"
+"  cdrecord write_track.\n"
+
+#: configupdate.c:493 configupdate.c:506
+msgid ""
+"This version of Gnometoaster is capable of calculating the\n"
+"Tracksize of an audio track created from a wav file accurately.\n"
+"To make use of this feature,the tracksize detection line\n"
+"of the wav file filetype registry entry needs to be changed.\n"
+msgstr ""
+"     \n"
+"     wav .\n"
+"   ,  \n"
+"      wav\n"
+"    .\n"
+
+#: configupdate.c:498 configupdate.c:511
+msgid ""
+"In order to fix a bug calculating the tracksize of an El Torito\n"
+"bootable filesystem,Gnometoaster needs to update the calculation\n"
+"rules stored in isotrack_tscalc.\n"
+msgstr ""
+"     \n"
+"    El Torito,\n"
+"    \n"
+"  isotrack_tscalc.\n"
+
+#: configupdate.c:503 configupdate.c:516
+msgid ""
+"This fix removes the -a option from the default mkisofs call\n"
+"to meet the requirements of recent mkisofs versions which would\n"
+"otherwise reject the call and create wrong track sizes for\n"
+"iso tracks.\n"
+"Also, several calls must be\n"
+"updated to match the new drive setup scheme.\n"
+"Apart from that, a bug recording/playing mono mp3s has been\n"
+"fixed in the default mpg123 call.\n"
+"Finally, from this version on gnometoaster can calculate the\n"
+"tracksize of mp3 audio tracks internally.\n"
+"Your configfile will be changed to facilitate this new\n"
+"functionality.\n"
+msgstr ""
+"    -a  \n"
+"mkisofs  ,   \n"
+"  mkisofs, ,   ,\n"
+"     \n"
+" ISO.\n"
+",     \n"
+"     .\n"
+" ,    \n"
+"/  mp3   mpg123\n"
+" .\n"
+",    ,  \n"
+"    mp3 .\n"
+"     \n"
+"   .\n"
+
+#: configupdate.c:516 configupdate.c:529
+msgid ""
+"We have do makde a few adjustments to the standard DAE call\n"
+"in Gnometoaster's preferences in order to support an additional\n"
+"field of Gnometoaster's drive setup scheme.\n"
+msgstr ""
+"    \n"
+"   DAE   \n"
+"     \n"
+"  .\n"
+
+#: configupdate.c:520 configupdate.c:533
+msgid ""
+"This version of Gnometoaster introduces the capability to store\n"
+"audio tracks to your harddrive as files. Gnometoaster will\n"
+"now adjust it's preferences to support storing audio tracks\n"
+"to your harddrive as either .wav or .mp3 files (where the latter\n"
+"is the default).\n"
+msgstr ""
+"    \n"
+"        \n"
+".      \n"
+"      \n"
+"  .wav  .mp3  (  ).\n"
+
+#: configupdate.c:526 configupdate.c:539
+msgid ""
+"This version fixes a wrong sox call for audiotrack->wav conversion.\n"
+"in your filetype registry.\n"
+msgstr ""
+"     sox \n"
+" \"-->wav-\"\n"
+"    .\n"
+
+#: configupdate.c:529 configupdate.c:542
+msgid ""
+"This version of Gnometoaster fixes a bug in the mountpoint autodetect\n"
+"expression of your recorder setup\n"
+"If you're running on a Big Endian system (e.g. PowerPC) I'll fix\n"
+"some parameters for the mp3 decoder to work properly\n"
+msgstr ""
+"      \n"
+"      \n"
+".\n"
+"        \n"
+"(. PowerPC),      \n"
+"  mp3.\n"
+
+#: configupdate.c:534 configupdate.c:547
+msgid ""
+"This version will fix broken .wav functionality on Big Endian Systems (e.g. "
+"PowerPC).\n"
+"It also removes the option to configure the filesystem destination area from "
+"your preferences.\n"
+"The symbolic link tree will be built in your temp directory from now on.\n"
+"Also note that the default cdrdao call will be changed to support CD-Text "
+"writing which implies making \"generic-mmc\" it's default driver\n"
+msgstr ""
+"    .wav     \n"
+"  (. PowerPC).  ,     \n"
+"    .   ,  \n"
+"      .\n"
+", ,   cdrdao        CD-,\n"
+"   \"generic-mmc\"      .\n"
+
+#: configupdate.c:540 configupdate.c:553
+msgid ""
+"Due to changes in the commandline parameters of mkisofs in recent releases\n"
+"Gnometoaster's tracksize detection scheme for ISO tracks needs to be "
+"changed.\n"
+msgstr ""
+"-      mkisofs   ,\n"
+"     ISO   .\n"
+
+#: configupdate.c:543 configupdate.c:556
+msgid ""
+"This version of GnomeToaster introduces two new filetypes supported by "
+"default:\n"
+"Ogg-Vorbis and Shorten. I will now create the corresponding filetype "
+"registry entries\n"
+"for those filetypes. If you want to actually use them, also make sure to "
+"install the commandline tools\n"
+"needed to decode them. GnomeToaster will tell you about the tools you need "
+"when you start using\n"
+"the new filetypes. (Note: In case you already have entries for .ogg or .shn "
+"in your filetypes registry, those entries will be left untouched).\n"
+"This version of GnomeToaster also adds native support for buffer-underrun "
+"protection schemes of all kind (burn-proof, just-link, ...) as far as "
+"supported by cdrecord."
+msgstr ""
+"          :\n"
+"Ogg-Vorbis  Shorten.           \n"
+".      ,    \n"
+"     .     \n"
+"       . (:    \n"
+"   .ogg  .shn    ,    ).\n"
+"         \n"
+"     (burn-proof, just-link, ...)   \n"
+" cdrecord."
+
+#: configupdate.c:552 configupdate.c:565
+msgid ""
+"This version of GnomeToaster will automatically set the title and artist ID3 "
+"tags\n"
+"of any mp3 files encoded through it using the mp3info tool.\n"
+"This requires an additional commandline call to be inserted into the mp3 "
+"encoder profile\n"
+"of GnomeToaster's internal filetypes registry.\n"
+"Do you want me to perform all necessary changes for you ?\n"
+msgstr ""
+"       \n"
+"     mp3     \n"
+",   mp3info.\n"
+"     mp3   \n"
+" ,      .\n"
+" ,      ?\n"
+
+#: configupdate.c:557 configupdate.c:570
+msgid ""
+"This version of GnomeToaster adapts to a new commandline scheme to be "
+"applied\n"
+"for ogg123, the ogg-vorbis file decoder.\n"
+"Answering this question with yes will break compatibility with\n"
+"versions of ogg123 older than 1.0rc2\n"
+msgstr ""
+"       \n"
+",   ogg123,   ogg-vorbis.\n"
+" \"\"   ,   \n"
+"  ogg123   1.0rc2\n"
+
+#: configupdate.c:564 configupdate.c:581
+msgid ""
+"You're running Gnometoaster for the first time.\n"
+"All Preferences have been set to default values.\n"
+"Make sure you change all settings to match your system!"
+msgstr ""
+"     .\n"
+"     .\n"
+"  ,     \n"
+"    ."
+
+#: configupdate.c:641 configupdate.c:659
+msgid "Configfile maintenance informations"
+msgstr "   "
+
+#: datacopydlg.c:123
+msgid "Throughput:"
+msgstr " :"
+
+#: datacopydlg.c:127
+msgid " kb/sec.\n"
+msgstr " /\n"
+
+#: datacopydlg.c:135
+msgid " elapsed\n"
+msgstr " \n"
+
+#: datacopydlg.c:147
+msgid " remaining\n"
+msgstr " \n"
+
+#: datacopydlg.c:150
+msgid "please wait...\n"
+msgstr ", ...\n"
+
+#: datacopydlg.c:172
+#, c-format
+msgid "(%.0f%% done) %s"
+msgstr "(%.0f%% ) %s"
+
+#: datacopydlg.c:287
+msgid "please wait..."
+msgstr ", ..."
+
+#: datacopydlg.c:296
+#, c-format
+msgid "%p%% done"
+msgstr "%p%% "
+
+#: datacopydlg.c:326
+msgid "Stop"
+msgstr ""
+
+#: datatrack.c:96
+#, c-format
+msgid ""
+"%s:\n"
+"Only the first data track of a CD can be used as a\n"
+"data source right now."
+msgstr ""
+"%s:\n"
+"        CD\n"
+"      ."
+
+#: datatrack.c:140
+#, c-format
+msgid "Datatrack #%2i of CD %#x"
+msgstr "  #%2i  CD %#x"
+
+#: datatrack.c:143
+#, c-format
+msgid "Datatrack #%02i of disc in drive %s"
+msgstr "  #%02i     %s"
+
+#: dependencies.c:43
+msgid "Debian/GNU Linux"
+msgstr "/  (Debian/GNU Linux)"
+
+#: dependencies.c:44
+msgid "The Debian Project"
+msgstr "  (The Debian Project)"
+
+#: dependencies.c:45
+msgid "potato"
+msgstr " (potato)"
+
+#: dependencies.c:47
+msgid ""
+"The ultimate, free Linux distribution. Not too easy for beginners, though."
+msgstr ""
+",   .\n"
+",     ."
+
+#: dependencies.c:66
+msgid "MPEG123 MPEG Audio Player"
+msgstr "MPEG123 MPEG  "
+
+#: dependencies.c:67
+msgid "Michael Hipp"
+msgstr "Michael Hipp"
+
+#: dependencies.c:68
+msgid "0.59r (1999/Jun/15)"
+msgstr "0.59r (15  1999)"
+
+#: dependencies.c:72
+msgid ""
+"MPG123 is used by Gnometoaster to decode MP3 files for both playing and "
+"recording. Essentially mpg123 is a commandline mpeg audio player capable of "
+"handling all sorts of mpeg audio streams (MPEG1+2, Layers 1 through 3)."
+msgstr ""
+"MPG123     MP3 ,   ,    .   , mpg123 --  mpeg     ,       mpeg (MPEG1+2,   "
+"1  3)."
+
+#: dependencies.c:94
+msgid "Sound Exchange"
+msgstr "Sound Exchange"
+
+#: dependencies.c:95
+msgid "Lance Norskog"
+msgstr "Lance Norskog"
+
+#: dependencies.c:96
+msgid "12.16"
+msgstr "12.16"
+
+#: dependencies.c:100
+msgid ""
+"Gnometoaster is using sox to convert various types of .wav files into cd "
+"digital audio streams suitable for direct recording.\n"
+"SoX is a sound file format converter for Unix and DOS PCs written by Lance "
+"Norskog and other invaluable contributors. It also does sample rate "
+"conversion and some sound effects. It's the swiss army knife of sound tools: "
+"the interface isn't great, but it does almost everything. SoX uses file "
+"suffices to determine the nature of a sound sample file. If it finds the "
+"suffix in its list, it uses the appropriate read or write handler to deal "
+"with that file. SoX has an auto-detect feature that attempts to figure out "
+"the nature of an unmarked sound sample. It works very well. This is the "
+"'auto' file format. (Source: http://www.spies.com/Sox/)."
+msgstr ""
+"   sox     .wav     CD     . SoX      Unix  DOS PC  Lance Norskog    . "
+"SoX         .     :  ,     . SoX        .       ,             . "
+"SoX          .    .    'auto'. (: http://www.spies.com/Sox/)."
+
+#: dependencies.c:124
+msgid "mkisofs"
+msgstr "mkisofs"
+
+#: dependencies.c:125
+msgid "Eric Youngdale/Yggdrasil Computing Inc."
+msgstr "Eric Youngdale/Yggdrasil Computing Inc."
+
+#: dependencies.c:126
+msgid "1.13"
+msgstr "1.13"
+
+#: dependencies.c:130
+msgid ""
+"Gnometoaster is using 'mkisofs' to create ISO9660 filesystems.\n"
+"That's what you usually do when creating a data CD (CDROM) with\n"
+"files on it.\n"
+"mkisofs is effectively a pre-mastering program to generate the iso9660 "
+"filesystem - it takes a snapshot of a\n"
+"given directory tree, and generates a binary image which will correspond to "
+"an iso9660 filesystem when written to a\n"
+"block device.(Source:mkisofs manpage (man 8 mkisofs))\n"
+msgstr ""
+"   'mkisofs'   \n"
+" ISO9660.  ,       \n"
+" (CDROM)    .\n"
+"  , mkisofs     \n"
+"  ISO9660 -       \n"
+"   ,    \n"
+" ISO9660     . \n"
+"(:   (man 8 mkisofs)). \n"
+
+#: dependencies.c:153
+msgid "cdda2wav"
+msgstr "cdda2wav"
+
+#: dependencies.c:154
+msgid "Heiko Eissfeldt (heiko@colossus.escape.de)"
+msgstr "Heiko Eissfeldt (heiko@colossus.escape.de)"
+
+#: dependencies.c:155 dependencies.c:211
+msgid "1.9"
+msgstr "1.9"
+
+#: dependencies.c:159
+msgid ""
+"'cdda2wav' is used by Gnometoaster to extract Digital Audio from\n"
+"audio compact discs for either encoding or burning.\n"
+"cdda2wav can retrieve audio tracks from CDROM drives (see\n"
+"[cdda2wav-]README  for  a list of drives) that are capable of reading\n"
+"audio data digitally to the host (CDDA). (Source:man cdda2wav)\n"
+msgstr ""
+" 'cdda2wav'    \n"
+"   -,   ,    .\n"
+"cdda2wav      CDROM \n"
+"(. [cdda2wav-]README   )   \n"
+"       (CDDA).\n"
+"(: man cdda2wav)\n"
+
+#: dependencies.c:175
+msgid "GoGo"
+msgstr "GoGo"
+
+#: dependencies.c:176
+msgid "PEN@MarineCat and shigeo"
+msgstr "PEN@MarineCat and shigeo"
+
+#: dependencies.c:177
+msgid "ver. 2.22 (Nov  8 1999)"
+msgstr "ver. 2.22 (Nov  8 1999)"
+
+#: dependencies.c:181
+msgid ""
+"Gnometoaster can use this program to encode audio tracks and .wav\n"
+"files into mp3 files.\n"
+"GoGo is an assembly optimized version of the free Lame encoder\n"
+"yielding about 6x encoding speed on my K6-3-400 machine.\n"
+"For many, Lame is considered the best free mp3 encoder available\n"
+"Note that in some countries it may be illegal to use this encoder\n"
+"because of software patent issues. See http://gnometoaster.rulez.org\n"
+"to find more informations about how you can protest against\n"
+"certain legal practices that may eventually make life a lot more\n"
+"difficult for us with intellectual property becoming more and more\n"
+"important an issue.\n"
+msgstr ""
+"      \n"
+"   .wav   mp3 .\n"
+"GoGo -  -   \n"
+" Lame,   6x \n"
+"   K6-3-400 . \n"
+"  Lame      \n"
+"mp3 .    ,   \n"
+"     \n"
+" -    .\n"
+"  http://gnometoaster.rulez.org   \n"
+"       \n"
+",      \n"
+"      \n"
+". \n"
+
+#: dependencies.c:209
+msgid "Cdrecord"
+msgstr "Cdrecord"
+
+#: dependencies.c:210
+msgid "Joerg Schilling (schilling@fokus.gmd.de)"
+msgstr "Joerg Schilling (schilling@fokus.gmd.de)"
+
+#: dependencies.c:215
+msgid ""
+"Gnometoaster depends on cdrecord for cd recording and CDRW blanking.\n"
+"Note that if cdrecord isn't installed, Gnometoaster may still be able\n"
+"to record discs in disc-at-once mode using cdrdao.\n"
+"Cdrecord is a commmandline cd recording program capable of using\n"
+"most cd recorders in track-at-once and some of them in disc-at-once mode\n"
+"(a rule of thumb would be to check wether your drive supports cuesheet "
+"recording).\n"
+"Please go to the cdrecord homepage to get a list of drives currently\n"
+"supported by this program. If your drive isn't supported, you can try\n"
+"using it with cdrdao, yet another backend Gnometoaster is supporting\n"
+msgstr ""
+"    cdrecord   CD   CDRW.\n"
+",   cdrecord  ,    \n"
+"      -- (DAO)\n"
+"  cdrdao.\n"
+"Cdrecord -     CD   , \n"
+"  CD    -- ,\n"
+"  ,   -- ( \n"
+" :        (cuesheet \n"
+"recording)).\n"
+",    cdrecord   \n"
+"      .    \n"
+" ,       cdrdao,\n"
+"    .\n"
+
+#: dependencies.c:241
+msgid "CdrDAO"
+msgstr "CdrDAO"
+
+#: dependencies.c:242
+msgid "Andreas Mueller <mueller@daneb.ping.de>"
+msgstr "Andreas Mueller <mueller@daneb.ping.de>"
+
+#: dependencies.c:243
+msgid "1.1.4"
+msgstr "1.1.4"
+
+#: dependencies.c:247
+msgid ""
+"Gnometoaster can use cdrdao to record CDRs in disc-at-once mode.\n"
+"Please make sure your drive is supported by cdrdao before trying\n"
+"to record a disc with this backend. As you don't have cdrdao\n"
+"installed, you can either turn off the \"Disc-at-once\" switch\n"
+"in Gnometoaster's recording panel or uncheck the \"Use alternative\n"
+"DAO client\" option in Gnometoaster's recording options.\n"
+"Gnometoaster will then use cdrecord to write your CDs\n"
+msgstr "           CDR.      ,      .       (Shorten)  http://www.etree.org.\n"
+
+#: dependencies.c:265
+msgid "Shorten Lossless Audio Compressor"
+msgstr "Shorten Lossless Audio Compressor"
+
+#: dependencies.c:266
+msgid "Tony Robinson and SoftSound Ltd"
+msgstr "Tony Robinson and SoftSound Ltd"
+
+#: dependencies.c:267
+msgid "version 3.2"
+msgstr "version 3.2"
+
+#: dependencies.c:271
+msgid ""
+"Gnometoaster can use shorten to burn .shn files to a CDR on-the-fly.The "
+"program can also be used to encode audio into .shn files.Shorten is a "
+"fileformat used for lossless audio compression and is therefore especially "
+"useful for musicians (e.g. for exchanging demo tapes).See http://www.etree."
+"org for details."
+msgstr ""
+"    shorten   .shn   CDR  .           .shn . Shorten -          , ,    "
+"(.    ).   http://www.etree.org  ."
+
+#: dependencies.c:281
+msgid "Shorten Lossless Audio Compression Tool"
+msgstr "Shorten Lossless Audio Compression Tool"
+
+#: dependencies.c:282
+msgid "Jason Jordan"
+msgstr "Jason Jordan"
+
+#: dependencies.c:283
+msgid "version 1.01"
+msgstr "version 1.01"
+
+#: dependencies.c:287
+msgid ""
+"GnomeToaster needs this program if you want to record shorten files to a "
+"CDR. It is used to calculate the exact space required by your shorten file "
+"on the medium.For informations about the Shorten format, visit http://www."
+"etree.org."
+msgstr "           CDR.      ,      .       (Shorten)  http://www.etree.org."
+
+#: dependencies.c:302
+msgid "Ogg-Vorbis commandline decoder "
+msgstr "Ogg-Vorbis commandline decoder "
+
+#: dependencies.c:303
+msgid "Kenneth Arnold"
+msgstr "Kenneth Arnold"
+
+#: dependencies.c:304
+msgid "version 1.0beta4"
+msgstr "version 1.0beta4"
+
+#: dependencies.c:308
+msgid ""
+"GnomeToaster needs this program to decode ogg-vorbis files (.ogg).Ogg-Vorbis "
+"is a totally free implementation of a lossy audio-compression scheme similar "
+"to Fraunhofer's mp3. It yields slightly better quality at the same bitrate "
+"and is free of any patented alghorithms."
+msgstr ""
+"       gg-vorbis (.ogg). Ogg-Vorbis -     -  ,   mp3.                 ."
+
+#: dependencies.c:323
+msgid "Ogg Vorbis File Information viewer"
+msgstr "Ogg Vorbis File Information viewer"
+
+#: dependencies.c:324
+msgid "James Atwill"
+msgstr "James Atwill"
+
+#: dependencies.c:325
+msgid "Jan 2001"
+msgstr "Jan 2001"
+
+#: dependencies.c:329
+msgid ""
+"GnomeToaster needs this program to determine the playing time of an ogg-"
+"vorbis file. Because it is not currently part of the main vorbis-tools "
+"distribution, a packaged version of the source can be downloaded from http://"
+"gnometoaster.rulez.org"
+msgstr ""
+"        ogg-vorbis .        vorbis-tools,        http://gnometoaster.rulez."
+"org"
+
+#: dependencies.c:445
+msgid "Dependency database"
+msgstr "   "
+
+#: dependencies.c:460
+#, c-format
+msgid ""
+"Gnometoaster failed to run the following command\n"
+"on your system:\n"
+"%s\n"
+msgstr ""
+"     \n"
+"  :\n"
+"%s\n"
+
+#: dialog.c:63 dialog.c:162 dialog.c:235 forms.c:148 menufunc.c:87
+#: dialog.c:215 dialog.c:239 menufunc.c:89
+msgid "OK"
+msgstr "OK"
+
+#: dialog.c:98 dialog.c:211 dialog.c:215
+msgid "YES"
+msgstr ""
+
+#: dialog.c:105 dialog.c:222 dialog.c:226
+msgid "NO"
+msgstr ""
+
+#: dialog.c:169 forms.c:157 preferences.c:414 preferences.c:432
+msgid "Cancel"
+msgstr ""
+
+#: filelist.c:29 filelist.c:30
+msgid "Filename"
+msgstr " "
+
+#: filelist.c:31 filelist.c:32
+msgid "Filesize"
+msgstr " "
+
+#: fileman.c:265
+msgid "Adding files to the filesystem"
+msgstr "    "
+
+#: fileman.c:273
+msgid "(please wait)"
+msgstr "(, )"
+
+#: fileman.c:377
+#, c-format
+msgid ""
+"GnomeToaster was unable to set tagging informations\n"
+"for the file it just encoded because the program responsible for doing this\n"
+"could not be found on your system:\n"
+"%s.\n"
+msgstr ""
+"      \n"
+"    ,   \n"
+"       :\n"
+"%s.\n"
+
+#: fileman.c:521
+msgid "Encoding File"
+msgstr " "
+
+#: fileman.c:563
+#, c-format
+msgid ""
+"No valid encoder could be found for the\n"
+"selected tracktype %s.\n"
+"Please create a corresponding entry in the\n"
+"encoder section of Gnometoaster's filetype\n"
+"registry.\n"
+msgstr ""
+"   \n"
+"    %s.\n"
+",   \n"
+"  ,     \n"
+".\n"
+
+#: fileman.c:637
+#, c-format
+msgid "Encode as %s"
+msgstr "  %s"
+
+#: fileman.c:713 fsedit.c:348 fsedit.c:549
+msgid "Create Directory"
+msgstr " "
+
+#: fileman.c:714
+msgid "New Directory"
+msgstr " "
+
+#: fileman.c:774
+msgid "Rename File"
+msgstr " "
+
+#: filetypes.c:167
+msgid "Select File"
+msgstr " "
+
+#: filetypes.c:265
+#, c-format
+msgid "Default for Tracktype %s"
+msgstr "    %s"
+
+#: filetypes.c:267
+msgid "Encoder settings"
+msgstr " "
+
+#: filetypes.c:271
+msgid "Encoder call"
+msgstr " "
+
+#: filetypes.c:275
+msgid "Encoder needs little Endian"
+msgstr "    "
+
+#: filetypes.c:283
+msgid "Set title info with"
+msgstr "  "
+
+#: filetypes.c:318 filetypes.c:418
+msgid "default"
+msgstr " "
+
+#: filetypes.c:342
+msgid "precache file before writing"
+msgstr "-   "
+
+#: filetypes.c:366 filetypes.c:384
+msgid "S"
+msgstr "S"
+
+#: filetypes.c:371 filetypes.c:389
+msgid "Select program"
+msgstr " "
+
+#: filetypes.c:402
+msgid "Encoding"
+msgstr ""
+
+#: filetypes.c:903
+msgid "Filetypes"
+msgstr " "
+
+#: filetypes.c:911
+msgid "Suffix"
+msgstr ""
+
+#: filetypes.c:917
+msgid "Tracktype"
+msgstr " "
+
+#: filetypes.c:923
+msgid "PC"
+msgstr ""
+
+#: filetypes.c:929
+msgid "Format"
+msgstr ""
+
+#: filetypes.c:935
+msgid "Filtername"
+msgstr " "
+
+#: filetypes.c:941 filetypes.c:953
+msgid " "
+msgstr " "
+
+#: filetypes.c:947
+msgid "Detect Size"
+msgstr " "
+
+#: filetypes.c:959
+msgid "Add Filetype"
+msgstr "  "
+
+#: fillstate.c:104
+#, c-format
+msgid "Fillstate %u MB/ %u MB ( %3.0f %% , %u:%02u/%u:%02u Minutes )"
+msgstr " %u / %u  ( %3.0f %% , %u:%02u/%u:%02u  )"
+
+#: fillstate.c:474
+msgid "Fillstate Setup"
+msgstr " "
+
+#: fillstate.c:477
+msgid "Use Themes"
+msgstr " "
+
+#: fillstate.c:498
+msgid "Add Color"
+msgstr " "
+
+#: fillstate.c:512
+msgid "Select Track Color"
+msgstr "  "
+
+#: forms.c:166 preferences.c:385 preferences.c:403
+msgid "Undo"
+msgstr ""
+
+#: fsedit.c:100
+#, c-format
+msgid ""
+"Error adding items to the Filesystem:\n"
+"%s\n"
+"Please check the result.\n"
+msgstr ""
+"      :\n"
+"%s\n"
+",  .\n"
+
+#: fsedit.c:107
+msgid "Unknown Filesystem error"
+msgstr "   "
+
+#: fsedit.c:347
+msgid "Delete File(s)"
+msgstr " ()"
+
+#: fsedit.c:347
+msgid "Delete marked file(s)/director(y/ies)"
+msgstr "  ()/()"
+
+#: fsedit.c:348
+msgid "Create a directory within the current file list"
+msgstr "     "
+
+#: fsedit.c:349
+msgid "Rename File(s)"
+msgstr " ()"
+
+#: fsedit.c:349
+msgid "Rename selected file(s)"
+msgstr "  ()"
+
+#: fsedit.c:350
+msgid "Make Boot Image"
+msgstr "  "
+
+#: fsedit.c:350
+msgid "Make current cd bootable using this boot image"
+msgstr "  CD     "
+
+#: fsedit.c:351
+msgid "Convert to.."
+msgstr " .."
+
+#: fsedit.c:351
+msgid "Convert this file to another filetype"
+msgstr "       "
+
+#: fsedit.c:520
+msgid "ISO Filesystem Volume ID"
+msgstr "    ISO"
+
+#: fsedit.c:548
+msgid "Delete Directory"
+msgstr " "
+
+#: fsedit.c:548
+msgid "Delete a directory"
+msgstr " "
+
+#: fsedit.c:549
+msgid "Create a directory within the currently selected dir"
+msgstr "    "
+
+#: fsedit.c:550
+msgid "Rename VOLUME/Directory"
+msgstr " /"
+
+#: fsedit.c:550
+msgid "Rename a directory"
+msgstr " "
+
+#: helpings.c:230
+#, c-format
+msgid "%02i hour(s) %02i min. %02i sec."
+msgstr "%02i () %02i . %02i ."
+
+#: isotrack.c:231
+msgid "Gnometoaster-Filesystem (ISO)"
+msgstr "-- (ISO)"
+
+#: isotrack.c:234
+msgid "Gnome Toaster"
+msgstr " "
+
+#: isotrack.c:294
+msgid "Create ISO Track with"
+msgstr " ISO "
+
+#: isotrack.c:299
+msgid "Multisession Options"
+msgstr " "
+
+#: isotrack.c:304
+msgid "Bootable Options"
+msgstr " "
+
+#: isotrack.c:309
+msgid "Use create call to determine tracksize"
+msgstr "      "
+
+#: isotrack.c:319
+msgid "Determine tracksize option"
+msgstr "   "
+
+#: isotrack.c:324
+msgid "Determine tracksize with"
+msgstr "   "
+
+#: isotrack.c:329
+msgid "Calculate tracksize as"
+msgstr "   "
+
+#: isotrack.c:334
+msgid "Mount an iso image with"
+msgstr " ISO  "
+
+#: isotrack.c:339
+msgid "Unmount an iso image with"
+msgstr " ISO  "
+
+#: isotrack.c:344
+msgid "Precache before writing"
+msgstr "-  "
+
+#: isotrack.c:354
+msgid "ISO Temporary files"
+msgstr "  ISO"
+
+#: menubar.c:27
+msgid "Import session"
+msgstr " "
+
+#: menubar.c:28 menubar.c:36
+msgid "Import the last session of this cd"
+msgstr "     CD"
+
+#: menubar.c:31
+msgid "Delete Filesystem"
+msgstr "  "
+
+#: menubar.c:32 menubar.c:40
+msgid "Delete the contents of fsedit"
+msgstr "  fsedit"
+
+#: menubar.c:41
+msgid "Refresh Tools list"
+msgstr "  "
+
+#: menubar.c:42 menubar.c:50
+msgid "scan for new plugins"
+msgstr "  "
+
+#: menubar.c:58
+msgid "Tools"
+msgstr ""
+
+#: menufunc.c:72 menufunc.c:74
+msgid "About Gnometoaster"
+msgstr " "
+
+#: precachetrack.c:201
+msgid " (precached)"
+msgstr " (-)"
+
+#: precachetrack.c:216
+msgid "precaching file..."
+msgstr "- ..."
+
+#: preferences.c:162 preferences.c:167
+#, c-format
+msgid ""
+"Gnometoaster failed to create it's symbolic link area\n"
+"which is essential for filesystem creation. Please make\n"
+"sure that the user running Gnometoaster has permission\n"
+"to create the following directory:\n"
+"%s\n"
+msgstr ""
+"      \n"
+",     \n"
+". ,   , \n"
+",  ,  \n"
+"   :\n"
+"%s\n"
+
+#: preferences.c:373 preferences.c:391
+msgid "GnomeToaster Preferences"
+msgstr " "
+
+#: preferences.c:394 preferences.c:412
+msgid "Apply"
+msgstr ""
+
+#: preferences.c:424 preferences.c:442
+msgid "Common"
+msgstr ""
+
+#: preferences.c:429 preferences.c:447
+msgid "Temp Files"
+msgstr " "
+
+#: preferences.c:434 preferences.c:452
+msgid "Precache Data Tracks:"
+msgstr "-  "
+
+#: preferences.c:440 preferences.c:458
+msgid "Use GNOME compatible drag and drop scheme"
+msgstr "      "
+
+#: preview.c:120
+#, c-format
+msgid ""
+"The selected audio driver couldn't be initialized\n"
+"Please make sure your sound hardware is configured\n"
+"properly, all drivers have been loaded and access permissions have\n"
+"been set properly.\n"
+"The error message as received from the Operating System was:\n"
+"'%s'\n"
+msgstr ""
+"     \n"
+". ,   ,\n"
+"     ,\n"
+"     \n"
+" .\n"
+"  ,   \n"
+", :\n"
+"%s\n"
+
+#: preview.c:294
+msgid "Drag files/tracks to be previewed here"
+msgstr "  /  "
+
+#: record.c:67
+msgid ""
+"\n"
+"Child exited unexpectedly."
+msgstr ""
+"\n"
+"   ."
+
+#: record.c:150
+msgid "CD recording process finished."
+msgstr "  CD ."
+
+#: record.c:242
+msgid ""
+"\n"
+"fixating CDROM."
+msgstr ""
+"\n"
+" CDROM."
+
+#: record.c:279
+msgid "fixating cdrom"
+msgstr " CDROM"
+
+#: record.c:351
+#, c-format
+msgid "Track %i/%i"
+msgstr " %i/%i"
+
+#: record.c:641
+msgid "Unnamed disc"
+msgstr " "
+
+#: record.c:642
+msgid "No Artist"
+msgstr " "
+
+#. done writing toc file
+#: record.c:687
+#, c-format
+msgid "Recording %i bytes to CD"
+msgstr " %i   CD"
+
+#: record.c:704
+msgid "Writing CD"
+msgstr " CD"
+
+#: record.c:714
+msgid "CD status"
+msgstr " CD"
+
+#: record.c:715
+msgid "Please wait"
+msgstr ", "
+
+#: record.c:727
+msgid "Track Status"
+msgstr " "
+
+#: record.c:763
+msgid "No recordable Tracks found."
+msgstr "    "
+
+#: record.c:767
+msgid "One or more tracks are unable to deliver their content."
+msgstr "        ."
+
+#: record.c:776
+msgid ""
+"Preview Player active,\n"
+"recording process cancelled."
+msgstr ""
+"  ,\n"
+"  ."
+
+#: record.c:811
+msgid "CDRW blanking complete."
+msgstr " CDRW ."
+
+#: record.c:857
+msgid "blanking CDRW."
+msgstr " CDRW."
+
+#: record.c:876
+msgid "Blanking CD/RW"
+msgstr " CD/RW"
+
+#. General recording options
+#: record.c:1029
+msgid "General"
+msgstr ""
+
+#: record.c:1032
+msgid "Recorder Speed"
+msgstr " "
+
+#: record.c:1043
+msgid "Eject disc"
+msgstr " "
+
+#: record.c:1054
+msgid "Reload disc"
+msgstr " "
+
+#: record.c:1065
+msgid "Dummy Write"
+msgstr " "
+
+#: record.c:1080
+msgid "Disc at once"
+msgstr "    (DAO)"
+
+#: record.c:1114
+msgid "CD-Text"
+msgstr "CD-"
+
+#. recording frame
+#: record.c:1151
+msgid "Recording"
+msgstr ""
+
+#: record.c:1155 toolbar.c:30 toolbar.c:31
+msgid "Record"
+msgstr ""
+
+#: record.c:1161
+msgid "Multisession"
+msgstr ""
+
+#: record.c:1172
+msgid "Buffer-Underrun Protection"
+msgstr "   "
+
+#: record.c:1192
+msgid "Fixate"
+msgstr ""
+
+#: record.c:1224 toolbar.c:27 toolbar.c:28
+msgid "Clear Disc"
+msgstr " "
+
+#: record.c:1230
+msgid "TOC only"
+msgstr " "
+
+#. common iso options
+#: record.c:1251
+msgid "ISO Options"
+msgstr " ISO"
+
+#: record.c:1254
+msgid "Boot Image"
+msgstr " "
+
+#: record.c:1270
+msgid "Make Bootable"
+msgstr " "
+
+#: record.c:1300
+msgid "Recorder"
+msgstr ""
+
+#: record.c:1305
+msgid "recording client"
+msgstr " "
+
+#: record.c:1309
+msgid "Use single call"
+msgstr "  "
+
+#: record.c:1317
+msgid "Single call FIFO"
+msgstr " (FIFO)  "
+
+#: record.c:1323
+msgid "Write whole CD with"
+msgstr "  CD "
+
+#: record.c:1327
+msgid "Track Item"
+msgstr "  "
+
+#: record.c:1334
+msgid "Fixating command"
+msgstr " "
+
+#: record.c:1339
+msgid "Trackwrite command"
+msgstr "  "
+
+#: record.c:1344
+msgid "Blank disc command"
+msgstr "  "
+
+#: record.c:1349
+msgid "Eject disc command"
+msgstr "  "
+
+#: record.c:1354
+msgid "Reload Disc with"
+msgstr "  "
+
+#: record.c:1359
+msgid "Multisession Info"
+msgstr "  "
+
+#: record.c:1365
+msgid "DAO Recording Client"
+msgstr "DAO- "
+
+#: record.c:1369
+msgid "Use alternative client"
+msgstr "  "
+
+#: record.c:1377
+msgid "DAO Write Command"
+msgstr " DAO-"
+
+#: record.c:1382 record.c:1392
+msgid "Scsi drive inquiry"
+msgstr " SCSI "
+
+#: record.c:1387 record.c:1397
+msgid "Get drive type"
+msgstr "  "
+
+#: record.c:1392 record.c:1402
+msgid "Get vendor name"
+msgstr "  "
+
+#: record.c:1397 record.c:1407
+msgid "Get drive identification"
+msgstr "  "
+
+#: record.c:1402 record.c:1412
+msgid "Get cdrom type"
+msgstr "  CDROM"
+
+#: record.c:1407 record.c:1417
+msgid "Mountpoint lookup file"
+msgstr "    "
+
+#: record.c:1412 record.c:1422
+msgid "Get mountpoint with"
+msgstr "  "
+
+#: rterm.c:58
+msgid "Client output"
+msgstr " "
+
+#: rterm.c:77
+msgid "GnomeToaster Recording Terminal"
+msgstr "  "
+
+#: selectlist.c:738 selectlist.c:739
+msgid "Select destination directory"
+msgstr "  "
+
+#: source.c:97 source.c:98
+msgid "Source Area"
+msgstr " "
+
+#. Main File selection Tree
+#: source.c:108 source.c:109
+msgid "Unix Tree"
+msgstr " UNIX"
+
+#: source.c:127 source.c:128
+msgid "Internal structures"
+msgstr " "
+
+#. CDROMs as source for track streams
+#: source.c:136 source.c:137
+msgid "CDROM drives"
+msgstr " CDROM"
+
+#: stdfiletrack.c:267
+#, c-format
+msgid ""
+"The tool necessary to detect the tracksize of\n"
+"files with the suffix '%s' couldn't be found on\n"
+"your system. In it's current configuration,\n"
+"GnomeToaster is trying to execute the following command\n"
+"to obtain the tracksize:\n"
+"%s\n"
+msgstr ""
+"   \n"
+"     '%s' \n"
+"    .   \n"
+"   \n"
+"     :\n"
+"%s\n"
+
+#: stdfiletrack.c:441
+#, c-format
+msgid ""
+"The Filter program for the \n"
+"suffix '%s' couldn't be found on your system.\n"
+"Please make sure your filetype registry\n"
+"is setup properly.\n"
+"Your current configuration is calling the following\n"
+"command to create the desired track:\n"
+"%s\n"
+msgstr ""
+"-   '%s'  \n"
+"  . , \n"
+" ,      \n"
+".\n"
+"     \n"
+"     :\n"
+"%s\n"
+
+#: toolbar.c:15 toolbar.c:16
+msgid "Quit"
+msgstr ""
+
+#: toolbar.c:15 toolbar.c:16
+msgid "Quit Application"
+msgstr "  "
+
+#: toolbar.c:19 toolbar.c:20
+msgid "Preferences..."
+msgstr "..."
+
+#: toolbar.c:19 toolbar.c:20
+msgid "Set Preferences"
+msgstr " "
+
+#: toolbar.c:23 toolbar.c:24
+msgid "Import"
+msgstr ""
+
+#: toolbar.c:23 toolbar.c:24
+msgid "Import the last session"
+msgstr "  "
+
+#: toolbar.c:27 toolbar.c:28
+msgid "Clear a CD-RW"
+msgstr " CD-RW"
+
+#: toolbar.c:30 toolbar.c:31
+msgid "Start recording"
+msgstr " "
+
+#: trackedit.c:39
+msgid "Delete Track"
+msgstr " "
+
+#: trackedit.c:39
+msgid "Remove selected track(s) from the tracklist"
+msgstr " () ()   "
+
+#: trackedit.c:40
+msgid "Encode as.."
+msgstr " .."
+
+#: trackedit.c:40
+msgid "Encode this object to a certain filetype"
+msgstr "      "
+
+#: trackedit.c:41
+msgid "Rename Track.."
+msgstr " .."
+
+#: trackedit.c:41
+msgid "Change Title, Artist and Name of a track"
+msgstr "  ,    "
+
+#: trackedit.c:58
+msgid "Rename track"
+msgstr " "
+
+#: trackedit.c:62
+msgid "Track Name"
+msgstr " "
+
+#: trackedit.c:66
+msgid "Title"
+msgstr ""
+
+#: trackedit.c:70
+msgid "Artist"
+msgstr ""
+
+#: trackedit.c:388
+msgid ""
+"You have used the Filesystem Editor but\n"
+"the ISO track is currently not in the list of tracks\n"
+"to be burned. Do you want me to add it for you ?\n"
+msgstr ""
+"    ,\n"
+"  ISO     \n"
+"     .\n"
+",      ?\n"
+
+#: trackedit.c:397
+msgid "Unnamed disc (Click title to set disc info)"
+msgstr "  (       )"
+
+#: trackedit.c:438
+msgid "Set disc informations"
+msgstr "   "
+
+#: trackedit.c:439
+msgid "Disc Title"
+msgstr " "
+
+#: trackedit.c:440
+msgid "Artist/Author"
+msgstr "/"
+
+#: tracklist.c:138
+#, c-format
+msgid "%s by %s"
+msgstr "%s - %s"
+
+#: tracklist.c:139
+#, c-format
+msgid "Medium by %s"
+msgstr " %s"
+
+#: tracklist.c:140
+#, c-format
+msgid "%s"
+msgstr "%s"
+
+#: tracklist.c:167
+msgid "Trackname"
+msgstr " "
+
+#: tracklist.c:202
+msgid "Tracksize"
+msgstr " "
+
+#: vfs.c:519
+msgid "Couldn't get sourcefile informations"
+msgstr "      "
+
+#: vfs.c:609
+msgid "Cannot copy/move directory into itself"
+msgstr "  /    "
+
+#: vfs.c:761
+msgid "Couldn't get file informations"
+msgstr "     "
+
+#: vfs_unixtree.c:360
+msgid "Can't add Symbolic link to a Symlinktree Filesystem"
+msgstr "         "
+
+#. report error
+#: vfs_unixtree.c:409
+msgid "Read/Write error copying file"
+msgstr " /   "
+
+#: cddrives.c:1101
+msgid "Choose recorder"
+msgstr ""
+
+#: configupdate.c:574
+msgid ""
+"This version of GnomeToaster will fix the recorder detection\n"
+"scheme to be able to properly autodetect some USB recorders.\n"
+"Thanks to Alec Flett for telling me about the problem and also\n"
+"providing a solution to it.\n"
+msgstr ""
+
+#: dialog.c:226
+msgid "CANCEL"
+msgstr ""
+
+#: menubar.c:17
+msgid "Home directory"
+msgstr ""
+
+#: menubar.c:18 toolbar.c:35
+msgid "Go to home directory"
+msgstr ""
+
+#: menubar.c:21
+msgid "Change _Recorder"
+msgstr ""
+
+#: menubar.c:22
+msgid "Change the currently used recorder"
+msgstr ""
+
+#: menubar.c:35
+msgid "_Import session"
+msgstr ""
+
+#: menubar.c:39
+msgid "_Delete Filesystem"
+msgstr ""
+
+#: menubar.c:49
+msgid "_Refresh Tools list"
+msgstr ""
+
+#: menubar.c:66
+msgid "_Tools"
+msgstr ""
+
+#: menusys.h:65
+msgid "About"
+msgstr ""
+
+#: menusys.h:65
+msgid "Program informations"
+msgstr ""
+
+#: menusys.h:66
+msgid "Exit"
+msgstr ""
+
+#: menusys.h:66
+msgid "Exit program"
+msgstr ""
+
+#: menusys.h:67
+msgid "Cut"
+msgstr ""
+
+#: menusys.h:67
+msgid "Cut selection"
+msgstr ""
+
+#: menusys.h:68
+msgid "Copy"
+msgstr ""
+
+#: menusys.h:68
+msgid "Copy selection"
+msgstr ""
+
+#: menusys.h:69
+msgid "Paste"
+msgstr ""
+
+#: menusys.h:69
+msgid "Paste selection"
+msgstr ""
+
+#: menusys.h:70
+msgid "Preferences"
+msgstr ""
+
+#: menusys.h:70
+msgid "Edit Preferences"
+msgstr ""
+
+#: menusys.h:71
+msgid "File"
+msgstr ""
+
+#: menusys.h:71
+msgid "File menu"
+msgstr ""
+
+#: menusys.h:72
+msgid "Edit menu"
+msgstr ""
+
+#: menusys.h:73
+msgid "Help"
+msgstr ""
+
+#: menusys.h:73
+msgid "Help menu"
+msgstr ""
+
+#: preferences.c:464
+msgid "Go to home directory at startup"
+msgstr ""
+
+#: record.c:222
+msgid "GnomeToaster is calling the following command:"
+msgstr ""
+
+#: record.c:1382
+msgid "Scsi bus scan command"
+msgstr ""
+
+#: record.c:1387
+msgid "Scsi parse scan output"
+msgstr ""
+
+#: toolbar.c:35
+msgid "Home"
+msgstr ""
